// @paypal/paypal-server-sdk/dist/esm/models/cardBrand.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { stringEnum } from '../schema';
/**
 * Enum for CardBrand
 */
export var CardBrand;
(function (CardBrand) {
    CardBrand["VISA"] = "VISA";
    CardBrand["MASTERCARD"] = "MASTERCARD";
    CardBrand["DISCOVER"] = "DISCOVER";
    CardBrand["AMEX"] = "AMEX";
    CardBrand["SOLO"] = "SOLO";
    CardBrand["JCB"] = "JCB";
    CardBrand["STAR"] = "STAR";
    CardBrand["DELTA"] = "DELTA";
    CardBrand["SWITCH"] = "SWITCH";
    CardBrand["MAESTRO"] = "MAESTRO";
    CardBrand["CBNATIONALE"] = "CB_NATIONALE";
    CardBrand["CONFIGOGA"] = "CONFIGOGA";
    CardBrand["CONFIDIS"] = "CONFIDIS";
    CardBrand["ELECTRON"] = "ELECTRON";
    CardBrand["CETELEM"] = "CETELEM";
    CardBrand["CHINAUNIONPAY"] = "CHINA_UNION_PAY";
    CardBrand["DINERS"] = "DINERS";
    CardBrand["ELO"] = "ELO";
    CardBrand["HIPER"] = "HIPER";
    CardBrand["HIPERCARD"] = "HIPERCARD";
    CardBrand["RUPAY"] = "RUPAY";
    CardBrand["GE"] = "GE";
    CardBrand["SYNCHRONY"] = "SYNCHRONY";
    CardBrand["EFTPOS"] = "EFTPOS";
    CardBrand["UNKNOWN"] = "UNKNOWN";
})(CardBrand || (CardBrand = {}));
/**
 * Schema for CardBrand
 */
export const cardBrandSchema = stringEnum(CardBrand, true);
