// @paypal/paypal-server-sdk/dist/esm/models/paymentSourceResponse.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { lazy, object, optional } from '../schema';
import { applePayPaymentObjectSchema, } from './applePayPaymentObject';
import { bancontactPaymentObjectSchema, } from './bancontactPaymentObject';
import { bLIKPaymentObjectSchema, } from './bLIKPaymentObject';
import { cardResponseSchema } from './cardResponse';
import { ePSPaymentObjectSchema } from './ePSPaymentObject';
import { giropayPaymentObjectSchema, } from './giropayPaymentObject';
import { googlePayWalletResponseSchema, } from './googlePayWalletResponse';
import { iDEALPaymentObjectSchema, } from './iDEALPaymentObject';
import { myBankPaymentObjectSchema, } from './myBankPaymentObject';
import { p24PaymentObjectSchema } from './p24PaymentObject';
import { payPalWalletResponseSchema, } from './payPalWalletResponse';
import { sofortPaymentObjectSchema, } from './sofortPaymentObject';
import { trustlyPaymentObjectSchema, } from './trustlyPaymentObject';
import { venmoWalletResponseSchema, } from './venmoWalletResponse';
export const paymentSourceResponseSchema = object({
    card: ['card', optional(lazy(() => cardResponseSchema))],
    paypal: ['paypal', optional(lazy(() => payPalWalletResponseSchema))],
    bancontact: [
        'bancontact',
        optional(lazy(() => bancontactPaymentObjectSchema)),
    ],
    blik: ['blik', optional(lazy(() => bLIKPaymentObjectSchema))],
    eps: ['eps', optional(lazy(() => ePSPaymentObjectSchema))],
    giropay: ['giropay', optional(lazy(() => giropayPaymentObjectSchema))],
    ideal: ['ideal', optional(lazy(() => iDEALPaymentObjectSchema))],
    mybank: ['mybank', optional(lazy(() => myBankPaymentObjectSchema))],
    p24: ['p24', optional(lazy(() => p24PaymentObjectSchema))],
    sofort: ['sofort', optional(lazy(() => sofortPaymentObjectSchema))],
    trustly: ['trustly', optional(lazy(() => trustlyPaymentObjectSchema))],
    applePay: ['apple_pay', optional(lazy(() => applePayPaymentObjectSchema))],
    googlePay: [
        'google_pay',
        optional(lazy(() => googlePayWalletResponseSchema)),
    ],
    venmo: ['venmo', optional(lazy(() => venmoWalletResponseSchema))],
});
