// @paypal/paypal-server-sdk/dist/esm/models/paymentSource.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { lazy, object, optional } from '../schema';
import { applePayRequestSchema } from './applePayRequest';
import { bancontactPaymentRequestSchema, } from './bancontactPaymentRequest';
import { bLIKPaymentRequestSchema, } from './bLIKPaymentRequest';
import { cardRequestSchema } from './cardRequest';
import { ePSPaymentRequestSchema, } from './ePSPaymentRequest';
import { giropayPaymentRequestSchema, } from './giropayPaymentRequest';
import { googlePayRequestSchema } from './googlePayRequest';
import { iDEALPaymentRequestSchema, } from './iDEALPaymentRequest';
import { myBankPaymentRequestSchema, } from './myBankPaymentRequest';
import { p24PaymentRequestSchema, } from './p24PaymentRequest';
import { payPalWalletSchema } from './payPalWallet';
import { sofortPaymentRequestSchema, } from './sofortPaymentRequest';
import { tokenSchema } from './token';
import { trustlyPaymentRequestSchema, } from './trustlyPaymentRequest';
import { venmoWalletRequestSchema, } from './venmoWalletRequest';
export const paymentSourceSchema = object({
    card: ['card', optional(lazy(() => cardRequestSchema))],
    token: ['token', optional(lazy(() => tokenSchema))],
    paypal: ['paypal', optional(lazy(() => payPalWalletSchema))],
    bancontact: [
        'bancontact',
        optional(lazy(() => bancontactPaymentRequestSchema)),
    ],
    blik: ['blik', optional(lazy(() => bLIKPaymentRequestSchema))],
    eps: ['eps', optional(lazy(() => ePSPaymentRequestSchema))],
    giropay: ['giropay', optional(lazy(() => giropayPaymentRequestSchema))],
    ideal: ['ideal', optional(lazy(() => iDEALPaymentRequestSchema))],
    mybank: ['mybank', optional(lazy(() => myBankPaymentRequestSchema))],
    p24: ['p24', optional(lazy(() => p24PaymentRequestSchema))],
    sofort: ['sofort', optional(lazy(() => sofortPaymentRequestSchema))],
    trustly: ['trustly', optional(lazy(() => trustlyPaymentRequestSchema))],
    applePay: ['apple_pay', optional(lazy(() => applePayRequestSchema))],
    googlePay: ['google_pay', optional(lazy(() => googlePayRequestSchema))],
    venmo: ['venmo', optional(lazy(() => venmoWalletRequestSchema))],
});
