// @paypal/paypal-server-sdk/dist/esm/models/payPalWallet.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { lazy, object, optional, string } from '../schema';
import { addressSchema } from './address';
import { nameSchema } from './name';
import { payPalWalletAttributesSchema, } from './payPalWalletAttributes';
import { payPalWalletExperienceContextSchema, } from './payPalWalletExperienceContext';
import { phoneWithTypeSchema } from './phoneWithType';
import { taxInfoSchema } from './taxInfo';
export const payPalWalletSchema = object({
    vaultId: ['vault_id', optional(string())],
    emailAddress: ['email_address', optional(string())],
    name: ['name', optional(lazy(() => nameSchema))],
    phone: ['phone', optional(lazy(() => phoneWithTypeSchema))],
    birthDate: ['birth_date', optional(string())],
    taxInfo: ['tax_info', optional(lazy(() => taxInfoSchema))],
    address: ['address', optional(lazy(() => addressSchema))],
    attributes: [
        'attributes',
        optional(lazy(() => payPalWalletAttributesSchema)),
    ],
    experienceContext: [
        'experience_context',
        optional(lazy(() => payPalWalletExperienceContextSchema)),
    ],
    billingAgreementId: ['billing_agreement_id', optional(string())],
});
