// @paypal/paypal-server-sdk/dist/esm/models/payPalWalletExperienceContext.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { object, optional, string } from '../schema';
import { payeePaymentMethodPreferenceSchema, } from './payeePaymentMethodPreference';
import { payPalExperienceLandingPageSchema, } from './payPalExperienceLandingPage';
import { payPalExperienceUserActionSchema, } from './payPalExperienceUserAction';
import { shippingPreferenceSchema, } from './shippingPreference';
export const payPalWalletExperienceContextSchema = object({
    brandName: ['brand_name', optional(string())],
    locale: ['locale', optional(string())],
    shippingPreference: [
        'shipping_preference',
        optional(shippingPreferenceSchema),
    ],
    returnUrl: ['return_url', optional(string())],
    cancelUrl: ['cancel_url', optional(string())],
    landingPage: ['landing_page', optional(payPalExperienceLandingPageSchema)],
    userAction: ['user_action', optional(payPalExperienceUserActionSchema)],
    paymentMethodPreference: [
        'payment_method_preference',
        optional(payeePaymentMethodPreferenceSchema),
    ],
});
