// @paypal/paypal-server-sdk/dist/esm/models/payPalWalletVaultInstruction.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { boolean, object, optional, string } from '../schema';
import { payPalPaymentTokenCustomerTypeSchema, } from './payPalPaymentTokenCustomerType';
import { payPalPaymentTokenUsagePatternSchema, } from './payPalPaymentTokenUsagePattern';
import { payPalPaymentTokenUsageTypeSchema, } from './payPalPaymentTokenUsageType';
import { storeInVaultInstructionSchema, } from './storeInVaultInstruction';
export const payPalWalletVaultInstructionSchema = object({
    storeInVault: ['store_in_vault', optional(storeInVaultInstructionSchema)],
    description: ['description', optional(string())],
    usagePattern: [
        'usage_pattern',
        optional(payPalPaymentTokenUsagePatternSchema),
    ],
    usageType: ['usage_type', payPalPaymentTokenUsageTypeSchema],
    customerType: [
        'customer_type',
        optional(payPalPaymentTokenCustomerTypeSchema),
    ],
    permitMultiplePaymentTokens: [
        'permit_multiple_payment_tokens',
        optional(boolean()),
    ],
});
