// @paypal/paypal-server-sdk/dist/esm/models/shipmentCarrier.js
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
import { stringEnum } from '../schema';
/**
 * Enum for ShipmentCarrier
 */
export var ShipmentCarrier;
(function (ShipmentCarrier) {
    ShipmentCarrier["DPDRU"] = "DPD_RU";
    ShipmentCarrier["BGBULGARIANPOST"] = "BG_BULGARIAN_POST";
    ShipmentCarrier["KRKOREAPOST"] = "KR_KOREA_POST";
    ShipmentCarrier["ZACOURIERIT"] = "ZA_COURIERIT";
    ShipmentCarrier["FREXAPAQ"] = "FR_EXAPAQ";
    ShipmentCarrier["AREEMIRATESPOST"] = "ARE_EMIRATES_POST";
    ShipmentCarrier["GAC"] = "GAC";
    ShipmentCarrier["GEIS"] = "GEIS";
    ShipmentCarrier["SFEX"] = "SF_EX";
    ShipmentCarrier["PAGO"] = "PAGO";
    ShipmentCarrier["MYHERMES"] = "MYHERMES";
    ShipmentCarrier["DIAMONDEUROGISTICS"] = "DIAMOND_EUROGISTICS";
    ShipmentCarrier["CORPORATECOURIERSWEBHOOK"] = "CORPORATECOURIERS_WEBHOOK";
    ShipmentCarrier["BOND"] = "BOND";
    ShipmentCarrier["OMNIPARCEL"] = "OMNIPARCEL";
    ShipmentCarrier["SKPOSTA"] = "SK_POSTA";
    ShipmentCarrier["PUROLATOR"] = "PUROLATOR";
    ShipmentCarrier["FETCHRWEBHOOK"] = "FETCHR_WEBHOOK";
    ShipmentCarrier["THEDELIVERYGROUP"] = "THEDELIVERYGROUP";
    ShipmentCarrier["CELLOSQUARE"] = "CELLO_SQUARE";
    ShipmentCarrier["TARRIVE"] = "TARRIVE";
    ShipmentCarrier["COLLIVERY"] = "COLLIVERY";
    ShipmentCarrier["MAINFREIGHT"] = "MAINFREIGHT";
    ShipmentCarrier["INDFIRSTFLIGHT"] = "IND_FIRSTFLIGHT";
    ShipmentCarrier["ACSWORLDWIDE"] = "ACSWORLDWIDE";
    ShipmentCarrier["AMSTAN"] = "AMSTAN";
    ShipmentCarrier["OKAYPARCEL"] = "OKAYPARCEL";
    ShipmentCarrier["ENVIALIAREFERENCE"] = "ENVIALIA_REFERENCE";
    ShipmentCarrier["SEURES"] = "SEUR_ES";
    ShipmentCarrier["CONTINENTAL"] = "CONTINENTAL";
    ShipmentCarrier["FDSEXPRESS"] = "FDSEXPRESS";
    ShipmentCarrier["AMAZONFBASWISHIP"] = "AMAZON_FBA_SWISHIP";
    ShipmentCarrier["WYNGS"] = "WYNGS";
    ShipmentCarrier["DHLACTIVETRACING"] = "DHL_ACTIVE_TRACING";
    ShipmentCarrier["ZYLLEM"] = "ZYLLEM";
    ShipmentCarrier["RUSTON"] = "RUSTON";
    ShipmentCarrier["XPOST"] = "XPOST";
    ShipmentCarrier["CORREOSES"] = "CORREOS_ES";
    ShipmentCarrier["DHLFR"] = "DHL_FR";
    ShipmentCarrier["PANASIA"] = "PAN_ASIA";
    ShipmentCarrier["BRTIT"] = "BRT_IT";
    ShipmentCarrier["SREKOREA"] = "SRE_KOREA";
    ShipmentCarrier["SPEEDEE"] = "SPEEDEE";
    ShipmentCarrier["TNTUK"] = "TNT_UK";
    ShipmentCarrier["VENIPAK"] = "VENIPAK";
    ShipmentCarrier["SHREENANDANCOURIER"] = "SHREENANDANCOURIER";
    ShipmentCarrier["CROSHOT"] = "CROSHOT";
    ShipmentCarrier["NIPOSTNG"] = "NIPOST_NG";
    ShipmentCarrier["EPSTGLBL"] = "EPST_GLBL";
    ShipmentCarrier["NEWGISTICS"] = "NEWGISTICS";
    ShipmentCarrier["POSTSLOVENIA"] = "POST_SLOVENIA";
    ShipmentCarrier["JERSEYPOST"] = "JERSEY_POST";
    ShipmentCarrier["BOMBINOEXP"] = "BOMBINOEXP";
    ShipmentCarrier["WMG"] = "WMG";
    ShipmentCarrier["XQEXPRESS"] = "XQ_EXPRESS";
    ShipmentCarrier["FURDECO"] = "FURDECO";
    ShipmentCarrier["LHTEXPRESS"] = "LHT_EXPRESS";
    ShipmentCarrier["SOUTHAFRICANPOSTOFFICE"] = "SOUTH_AFRICAN_POST_OFFICE";
    ShipmentCarrier["SPOTON"] = "SPOTON";
    ShipmentCarrier["DIMERCO"] = "DIMERCO";
    ShipmentCarrier["CYPRUSPOSTCYP"] = "CYPRUS_POST_CYP";
    ShipmentCarrier["ABCUSTOM"] = "ABCUSTOM";
    ShipmentCarrier["INDDELIVREE"] = "IND_DELIVREE";
    ShipmentCarrier["CNBESTEXPRESS"] = "CN_BESTEXPRESS";
    ShipmentCarrier["DXSFTP"] = "DX_SFTP";
    ShipmentCarrier["PICKUPPMYS"] = "PICKUPP_MYS";
    ShipmentCarrier["FMX"] = "FMX";
    ShipmentCarrier["HELLMANN"] = "HELLMANN";
    ShipmentCarrier["SHIPITASIA"] = "SHIP_IT_ASIA";
    ShipmentCarrier["KERRYECOMMERCE"] = "KERRY_ECOMMERCE";
    ShipmentCarrier["FRETERAPIDO"] = "FRETERAPIDO";
    ShipmentCarrier["PITNEYBOWES"] = "PITNEY_BOWES";
    ShipmentCarrier["XPRESSENDK"] = "XPRESSEN_DK";
    ShipmentCarrier["SEURSPAPI"] = "SEUR_SP_API";
    ShipmentCarrier["DELIVERYONTIME"] = "DELIVERYONTIME";
    ShipmentCarrier["JINSUNG"] = "JINSUNG";
    ShipmentCarrier["TRANSKARGO"] = "TRANS_KARGO";
    ShipmentCarrier["SWISHIPDE"] = "SWISHIP_DE";
    ShipmentCarrier["IVOYWEBHOOK"] = "IVOY_WEBHOOK";
    ShipmentCarrier["AIRMEEWEBHOOK"] = "AIRMEE_WEBHOOK";
    ShipmentCarrier["DHLBENELUX"] = "DHL_BENELUX";
    ShipmentCarrier["FIRSTMILE"] = "FIRSTMILE";
    ShipmentCarrier["FASTWAYIR"] = "FASTWAY_IR";
    ShipmentCarrier["HHEXP"] = "HH_EXP";
    ShipmentCarrier["MYSMYPOSTONLINE"] = "MYS_MYPOST_ONLINE";
    ShipmentCarrier["TNTNL"] = "TNT_NL";
    ShipmentCarrier["TIPSA"] = "TIPSA";
    ShipmentCarrier["TAQBINMY"] = "TAQBIN_MY";
    ShipmentCarrier["KGMHUB"] = "KGMHUB";
    ShipmentCarrier["INTEXPRESS"] = "INTEXPRESS";
    ShipmentCarrier["OVERSEEXP"] = "OVERSE_EXP";
    ShipmentCarrier["ONECLICK"] = "ONECLICK";
    ShipmentCarrier["ROADRUNNERFREIGHT"] = "ROADRUNNER_FREIGHT";
    ShipmentCarrier["GLSCROTIA"] = "GLS_CROTIA";
    ShipmentCarrier["MRWFTP"] = "MRW_FTP";
    ShipmentCarrier["BLUEX"] = "BLUEX";
    ShipmentCarrier["DYLT"] = "DYLT";
    ShipmentCarrier["DPDIR"] = "DPD_IR";
    ShipmentCarrier["SINGLBL"] = "SIN_GLBL";
    ShipmentCarrier["TUFFNELLSREFERENCE"] = "TUFFNELLS_REFERENCE";
    ShipmentCarrier["CJPACKET"] = "CJPACKET";
    ShipmentCarrier["MILKMAN"] = "MILKMAN";
    ShipmentCarrier["ASIGNA"] = "ASIGNA";
    ShipmentCarrier["ONEWORLDEXPRESS"] = "ONEWORLDEXPRESS";
    ShipmentCarrier["ROYALMAIL"] = "ROYAL_MAIL";
    ShipmentCarrier["VIAEXPRESS"] = "VIA_EXPRESS";
    ShipmentCarrier["TIGFREIGHT"] = "TIGFREIGHT";
    ShipmentCarrier["ZTOEXPRESS"] = "ZTO_EXPRESS";
    ShipmentCarrier["TWOGO"] = "TWO_GO";
    ShipmentCarrier["IML"] = "IML";
    ShipmentCarrier["INTELVALLEY"] = "INTEL_VALLEY";
    ShipmentCarrier["EFS"] = "EFS";
    ShipmentCarrier["UKUKMAIL"] = "UK_UK_MAIL";
    ShipmentCarrier["RAM"] = "RAM";
    ShipmentCarrier["ALLIEDEXPRESS"] = "ALLIEDEXPRESS";
    ShipmentCarrier["APCOVERNIGHT"] = "APC_OVERNIGHT";
    ShipmentCarrier["SHIPPIT"] = "SHIPPIT";
    ShipmentCarrier["TFM"] = "TFM";
    ShipmentCarrier["MXPRESS"] = "M_XPRESS";
    ShipmentCarrier["HDBBOX"] = "HDB_BOX";
    ShipmentCarrier["CLEVYLINKS"] = "CLEVY_LINKS";
    ShipmentCarrier["IBEONE"] = "IBEONE";
    ShipmentCarrier["FIEGENL"] = "FIEGE_NL";
    ShipmentCarrier["KWEGLOBAL"] = "KWE_GLOBAL";
    ShipmentCarrier["CTCEXPRESS"] = "CTC_EXPRESS";
    ShipmentCarrier["LAOPOST"] = "LAO_POST";
    ShipmentCarrier["AMAZON"] = "AMAZON";
    ShipmentCarrier["MORELINK"] = "MORE_LINK";
    ShipmentCarrier["JX"] = "JX";
    ShipmentCarrier["EASYMAIL"] = "EASY_MAIL";
    ShipmentCarrier["ADUIEPYLE"] = "ADUIEPYLE";
    ShipmentCarrier["GBPANTHER"] = "GB_PANTHER";
    ShipmentCarrier["EXPRESSSALE"] = "EXPRESSSALE";
    ShipmentCarrier["SGDETRACK"] = "SG_DETRACK";
    ShipmentCarrier["TRUNKRSWEBHOOK"] = "TRUNKRS_WEBHOOK";
    ShipmentCarrier["MATDESPATCH"] = "MATDESPATCH";
    ShipmentCarrier["DICOM"] = "DICOM";
    ShipmentCarrier["MBW"] = "MBW";
    ShipmentCarrier["KHMCAMBODIAPOST"] = "KHM_CAMBODIA_POST";
    ShipmentCarrier["SINOTRANS"] = "SINOTRANS";
    ShipmentCarrier["BRTITPARCELID"] = "BRT_IT_PARCELID";
    ShipmentCarrier["DHLSUPPLYCHAIN"] = "DHL_SUPPLY_CHAIN";
    ShipmentCarrier["DHLPL"] = "DHL_PL";
    ShipmentCarrier["TOPYOU"] = "TOPYOU";
    ShipmentCarrier["PALEXPRESS"] = "PALEXPRESS";
    ShipmentCarrier["DHLSG"] = "DHL_SG";
    ShipmentCarrier["CNWEDO"] = "CN_WEDO";
    ShipmentCarrier["FULFILLME"] = "FULFILLME";
    ShipmentCarrier["DPDDELISTRACK"] = "DPD_DELISTRACK";
    ShipmentCarrier["UPSREFERENCE"] = "UPS_REFERENCE";
    ShipmentCarrier["CARIBOU"] = "CARIBOU";
    ShipmentCarrier["LOCUSWEBHOOK"] = "LOCUS_WEBHOOK";
    ShipmentCarrier["DSV"] = "DSV";
    ShipmentCarrier["CNGOFLY"] = "CN_GOFLY";
    ShipmentCarrier["P2PTRC"] = "P2P_TRC";
    ShipmentCarrier["DIRECTPARCELS"] = "DIRECTPARCELS";
    ShipmentCarrier["NOVAPOSHTAINT"] = "NOVA_POSHTA_INT";
    ShipmentCarrier["FEDEXPOLAND"] = "FEDEX_POLAND";
    ShipmentCarrier["CNJCEX"] = "CN_JCEX";
    ShipmentCarrier["FARINTERNATIONAL"] = "FAR_INTERNATIONAL";
    ShipmentCarrier["IDEXPRESS"] = "IDEXPRESS";
    ShipmentCarrier["GANGBAO"] = "GANGBAO";
    ShipmentCarrier["NEWAY"] = "NEWAY";
    ShipmentCarrier["POSTNLINT3S"] = "POSTNL_INT_3_S";
    ShipmentCarrier["RPXID"] = "RPX_ID";
    ShipmentCarrier["DESIGNERTRANSPORTWEBHOOK"] = "DESIGNERTRANSPORT_WEBHOOK";
    ShipmentCarrier["GLSSLOVEN"] = "GLS_SLOVEN";
    ShipmentCarrier["PARCELLEDIN"] = "PARCELLED_IN";
    ShipmentCarrier["GSIEXPRESS"] = "GSI_EXPRESS";
    ShipmentCarrier["CONWAY"] = "CON_WAY";
    ShipmentCarrier["BROUWERTRANSPORT"] = "BROUWER_TRANSPORT";
    ShipmentCarrier["CPEX"] = "CPEX";
    ShipmentCarrier["ISRAELPOST"] = "ISRAEL_POST";
    ShipmentCarrier["DTDCIN"] = "DTDC_IN";
    ShipmentCarrier["PTTPOST"] = "PTT_POST";
    ShipmentCarrier["XDEWEBHOOK"] = "XDE_WEBHOOK";
    ShipmentCarrier["TOLOS"] = "TOLOS";
    ShipmentCarrier["GIAOHANG"] = "GIAO_HANG";
    ShipmentCarrier["GEODISESPACE"] = "GEODIS_ESPACE";
    ShipmentCarrier["MAGYARHU"] = "MAGYAR_HU";
    ShipmentCarrier["DOORDASHWEBHOOK"] = "DOORDASH_WEBHOOK";
    ShipmentCarrier["TIKIID"] = "TIKI_ID";
    ShipmentCarrier["CJHKINTERNATIONAL"] = "CJ_HK_INTERNATIONAL";
    ShipmentCarrier["STARTRACKEXPRESS"] = "STAR_TRACK_EXPRESS";
    ShipmentCarrier["HELTHJEM"] = "HELTHJEM";
    ShipmentCarrier["SFB2C"] = "SFB2C";
    ShipmentCarrier["FREIGHTQUOTE"] = "FREIGHTQUOTE";
    ShipmentCarrier["LANDMARKGLOBALREFERENCE"] = "LANDMARK_GLOBAL_REFERENCE";
    ShipmentCarrier["PARCEL2GO"] = "PARCEL2GO";
    ShipmentCarrier["DELNEXT"] = "DELNEXT";
    ShipmentCarrier["RCL"] = "RCL";
    ShipmentCarrier["CGSEXPRESS"] = "CGS_EXPRESS";
    ShipmentCarrier["HKPOST"] = "HK_POST";
    ShipmentCarrier["SAPEXPRESS"] = "SAP_EXPRESS";
    ShipmentCarrier["PARCELPOSTSG"] = "PARCELPOST_SG";
    ShipmentCarrier["HERMES"] = "HERMES";
    ShipmentCarrier["INDSAFEEXPRESS"] = "IND_SAFEEXPRESS";
    ShipmentCarrier["TOPHATTEREXPRESS"] = "TOPHATTEREXPRESS";
    ShipmentCarrier["MGLOBAL"] = "MGLOBAL";
    ShipmentCarrier["AVERITT"] = "AVERITT";
    ShipmentCarrier["LEADER"] = "LEADER";
    ShipmentCarrier["_2EBOX"] = "_2EBOX";
    ShipmentCarrier["SGSPEEDPOST"] = "SG_SPEEDPOST";
    ShipmentCarrier["DBSCHENKERSE"] = "DBSCHENKER_SE";
    ShipmentCarrier["ISRPOSTDOMESTIC"] = "ISR_POST_DOMESTIC";
    ShipmentCarrier["BESTWAYPARCEL"] = "BESTWAYPARCEL";
    ShipmentCarrier["ASENDIADE"] = "ASENDIA_DE";
    ShipmentCarrier["NIGHTLINEUK"] = "NIGHTLINE_UK";
    ShipmentCarrier["TAQBINSG"] = "TAQBIN_SG";
    ShipmentCarrier["TCKEXPRESS"] = "TCK_EXPRESS";
    ShipmentCarrier["ENDEAVOURDELIVERY"] = "ENDEAVOUR_DELIVERY";
    ShipmentCarrier["NANJINGWOYUAN"] = "NANJINGWOYUAN";
    ShipmentCarrier["HEPPNERFR"] = "HEPPNER_FR";
    ShipmentCarrier["EMPSCN"] = "EMPS_CN";
    ShipmentCarrier["FONSEN"] = "FONSEN";
    ShipmentCarrier["PICKRR"] = "PICKRR";
    ShipmentCarrier["APCOVERNIGHTCONNUM"] = "APC_OVERNIGHT_CONNUM";
    ShipmentCarrier["STARTRACKNEXTFLIGHT"] = "STAR_TRACK_NEXT_FLIGHT";
    ShipmentCarrier["DAJIN"] = "DAJIN";
    ShipmentCarrier["UPSFREIGHT"] = "UPS_FREIGHT";
    ShipmentCarrier["POSTAPLUS"] = "POSTA_PLUS";
    ShipmentCarrier["CEVA"] = "CEVA";
    ShipmentCarrier["ANSERX"] = "ANSERX";
    ShipmentCarrier["JSEXPRESS"] = "JS_EXPRESS";
    ShipmentCarrier["PADTF"] = "PADTF";
    ShipmentCarrier["UPSMAILINNOVATIONS"] = "UPS_MAIL_INNOVATIONS";
    ShipmentCarrier["EZSHIP"] = "EZSHIP";
    ShipmentCarrier["SYPOST"] = "SYPOST";
    ShipmentCarrier["AMAZONSHIPMCF"] = "AMAZON_SHIP_MCF";
    ShipmentCarrier["YUSEN"] = "YUSEN";
    ShipmentCarrier["BRING"] = "BRING";
    ShipmentCarrier["SDAIT"] = "SDA_IT";
    ShipmentCarrier["GBA"] = "GBA";
    ShipmentCarrier["NEWEGGEXPRESS"] = "NEWEGGEXPRESS";
    ShipmentCarrier["SPEEDCOURIERSGR"] = "SPEEDCOURIERS_GR";
    ShipmentCarrier["FORRUN"] = "FORRUN";
    ShipmentCarrier["PICKUP"] = "PICKUP";
    ShipmentCarrier["ECMS"] = "ECMS";
    ShipmentCarrier["INTELIPOST"] = "INTELIPOST";
    ShipmentCarrier["FLASHEXPRESS"] = "FLASHEXPRESS";
    ShipmentCarrier["CNSTO"] = "CN_STO";
    ShipmentCarrier["SEKOSFTP"] = "SEKO_SFTP";
    ShipmentCarrier["HOMEDELIVERYSOLUTIONS"] = "HOME_DELIVERY_SOLUTIONS";
    ShipmentCarrier["DPDHGRY"] = "DPD_HGRY";
    ShipmentCarrier["KERRYTTCVN"] = "KERRYTTC_VN";
    ShipmentCarrier["JOYINGBOX"] = "JOYING_BOX";
    ShipmentCarrier["TOTALEXPRESS"] = "TOTAL_EXPRESS";
    ShipmentCarrier["ZJSEXPRESS"] = "ZJS_EXPRESS";
    ShipmentCarrier["STARKEN"] = "STARKEN";
    ShipmentCarrier["DEMANDSHIP"] = "DEMANDSHIP";
    ShipmentCarrier["CNDPEX"] = "CN_DPEX";
    ShipmentCarrier["AUPOSTCN"] = "AUPOST_CN";
    ShipmentCarrier["LOGISTERS"] = "LOGISTERS";
    ShipmentCarrier["GOGLOBALPOST"] = "GOGLOBALPOST";
    ShipmentCarrier["GLSCZ"] = "GLS_CZ";
    ShipmentCarrier["PAACKWEBHOOK"] = "PAACK_WEBHOOK";
    ShipmentCarrier["GRABWEBHOOK"] = "GRAB_WEBHOOK";
    ShipmentCarrier["PARCELPOINT"] = "PARCELPOINT";
    ShipmentCarrier["ICUMULUS"] = "ICUMULUS";
    ShipmentCarrier["DAIGLOBALTRACK"] = "DAIGLOBALTRACK";
    ShipmentCarrier["GLOBALIPARCEL"] = "GLOBAL_IPARCEL";
    ShipmentCarrier["YURTICIKARGO"] = "YURTICI_KARGO";
    ShipmentCarrier["CNPAYPALPACKAGE"] = "CN_PAYPAL_PACKAGE";
    ShipmentCarrier["PARCEL2POST"] = "PARCEL_2_POST";
    ShipmentCarrier["GLSIT"] = "GLS_IT";
    ShipmentCarrier["PILLOGISTICS"] = "PIL_LOGISTICS";
    ShipmentCarrier["HEPPNER"] = "HEPPNER";
    ShipmentCarrier["GENERALOVERNIGHT"] = "GENERAL_OVERNIGHT";
    ShipmentCarrier["HAPPY2POINT"] = "HAPPY2POINT";
    ShipmentCarrier["CHITCHATS"] = "CHITCHATS";
    ShipmentCarrier["SMOOTH"] = "SMOOTH";
    ShipmentCarrier["CLELOGISTICS"] = "CLE_LOGISTICS";
    ShipmentCarrier["FIEGE"] = "FIEGE";
    ShipmentCarrier["MXCARGO"] = "MX_CARGO";
    ShipmentCarrier["ZIINGFINALMILE"] = "ZIINGFINALMILE";
    ShipmentCarrier["DAYTONFREIGHT"] = "DAYTON_FREIGHT";
    ShipmentCarrier["TCS"] = "TCS";
    ShipmentCarrier["AEX"] = "AEX";
    ShipmentCarrier["HERMESDE"] = "HERMES_DE";
    ShipmentCarrier["ROUTIFICWEBHOOK"] = "ROUTIFIC_WEBHOOK";
    ShipmentCarrier["GLOBAVEND"] = "GLOBAVEND";
    ShipmentCarrier["CJLOGISTICS"] = "CJ_LOGISTICS";
    ShipmentCarrier["PALLETNETWORK"] = "PALLET_NETWORK";
    ShipmentCarrier["RAFPH"] = "RAF_PH";
    ShipmentCarrier["UKXDP"] = "UK_XDP";
    ShipmentCarrier["PAPEREXPRESS"] = "PAPER_EXPRESS";
    ShipmentCarrier["LAPOSTESUIVI"] = "LA_POSTE_SUIVI";
    ShipmentCarrier["PAQUETEXPRESS"] = "PAQUETEXPRESS";
    ShipmentCarrier["LIEFERY"] = "LIEFERY";
    ShipmentCarrier["STRECKTRANSPORT"] = "STRECK_TRANSPORT";
    ShipmentCarrier["PONYEXPRESS"] = "PONY_EXPRESS";
    ShipmentCarrier["ALWAYSEXPRESS"] = "ALWAYS_EXPRESS";
    ShipmentCarrier["GBSBROKER"] = "GBS_BROKER";
    ShipmentCarrier["CITYLINKMY"] = "CITYLINK_MY";
    ShipmentCarrier["ALLJOY"] = "ALLJOY";
    ShipmentCarrier["YODEL"] = "YODEL";
    ShipmentCarrier["YODELDIR"] = "YODEL_DIR";
    ShipmentCarrier["STONE3PL"] = "STONE3PL";
    ShipmentCarrier["PARCELPALWEBHOOK"] = "PARCELPAL_WEBHOOK";
    ShipmentCarrier["DHLECOMERCEASA"] = "DHL_ECOMERCE_ASA";
    ShipmentCarrier["SIMPLYPOST"] = "SIMPLYPOST";
    ShipmentCarrier["KYEXPRESS"] = "KY_EXPRESS";
    ShipmentCarrier["SHENZHEN"] = "SHENZHEN";
    ShipmentCarrier["USLASERSHIP"] = "US_LASERSHIP";
    ShipmentCarrier["UCEXPRE"] = "UC_EXPRE";
    ShipmentCarrier["DIDADI"] = "DIDADI";
    ShipmentCarrier["CJKR"] = "CJ_KR";
    ShipmentCarrier["DBSCHENKERB2B"] = "DBSCHENKER_B2B";
    ShipmentCarrier["MXE"] = "MXE";
    ShipmentCarrier["CAEDELIVERS"] = "CAE_DELIVERS";
    ShipmentCarrier["PFCEXPRESS"] = "PFCEXPRESS";
    ShipmentCarrier["WHISTL"] = "WHISTL";
    ShipmentCarrier["WEPOST"] = "WEPOST";
    ShipmentCarrier["DHLPARCELES"] = "DHL_PARCEL_ES";
    ShipmentCarrier["DDEXPRESS"] = "DDEXPRESS";
    ShipmentCarrier["ARAMEXAU"] = "ARAMEX_AU";
    ShipmentCarrier["BNEED"] = "BNEED";
    ShipmentCarrier["HKTGX"] = "HK_TGX";
    ShipmentCarrier["LATVIJASPASTS"] = "LATVIJAS_PASTS";
    ShipmentCarrier["VIAEUROPE"] = "VIAEUROPE";
    ShipmentCarrier["CORREOUY"] = "CORREO_UY";
    ShipmentCarrier["CHRONOPOSTFR"] = "CHRONOPOST_FR";
    ShipmentCarrier["JNET"] = "J_NET";
    ShipmentCarrier["_6LS"] = "_6LS";
    ShipmentCarrier["BLRBELPOST"] = "BLR_BELPOST";
    ShipmentCarrier["BIRDSYSTEM"] = "BIRDSYSTEM";
    ShipmentCarrier["DOBROPOST"] = "DOBROPOST";
    ShipmentCarrier["WAHANAID"] = "WAHANA_ID";
    ShipmentCarrier["WEASHIP"] = "WEASHIP";
    ShipmentCarrier["SONICTL"] = "SONICTL";
    ShipmentCarrier["KWT"] = "KWT";
    ShipmentCarrier["AFLLOGFTP"] = "AFLLOG_FTP";
    ShipmentCarrier["SKYNETWORLDWIDE"] = "SKYNET_WORLDWIDE";
    ShipmentCarrier["NOVAPOSHTA"] = "NOVA_POSHTA";
    ShipmentCarrier["SEINO"] = "SEINO";
    ShipmentCarrier["SZENDEX"] = "SZENDEX";
    ShipmentCarrier["BPOSTINT"] = "BPOST_INT";
    ShipmentCarrier["DBSCHENKERSV"] = "DBSCHENKER_SV";
    ShipmentCarrier["AODEUTSCHLAND"] = "AO_DEUTSCHLAND";
    ShipmentCarrier["EUFLEETSOLUTIONS"] = "EU_FLEET_SOLUTIONS";
    ShipmentCarrier["PCFCORP"] = "PCFCORP";
    ShipmentCarrier["LINKBRIDGE"] = "LINKBRIDGE";
    ShipmentCarrier["PRIMAMULTICIPTA"] = "PRIMAMULTICIPTA";
    ShipmentCarrier["COUREX"] = "COUREX";
    ShipmentCarrier["ZAJILEXPRESS"] = "ZAJIL_EXPRESS";
    ShipmentCarrier["COLLECTCO"] = "COLLECTCO";
    ShipmentCarrier["JTEXPRESS"] = "JTEXPRESS";
    ShipmentCarrier["FEDEXUK"] = "FEDEX_UK";
    ShipmentCarrier["USHIP"] = "USHIP";
    ShipmentCarrier["PIXSELL"] = "PIXSELL";
    ShipmentCarrier["SHIPTOR"] = "SHIPTOR";
    ShipmentCarrier["CDEK"] = "CDEK";
    ShipmentCarrier["VNMVIETTELPOST"] = "VNM_VIETTELPOST";
    ShipmentCarrier["CJCENTURY"] = "CJ_CENTURY";
    ShipmentCarrier["GSO"] = "GSO";
    ShipmentCarrier["VIWO"] = "VIWO";
    ShipmentCarrier["SKYBOX"] = "SKYBOX";
    ShipmentCarrier["KERRYTJ"] = "KERRYTJ";
    ShipmentCarrier["NTLOGISTICSVN"] = "NTLOGISTICS_VN";
    ShipmentCarrier["SDHSCM"] = "SDH_SCM";
    ShipmentCarrier["ZINC"] = "ZINC";
    ShipmentCarrier["DPESOUTHAFRC"] = "DPE_SOUTH_AFRC";
    ShipmentCarrier["CESKACZ"] = "CESKA_CZ";
    ShipmentCarrier["ACSGR"] = "ACS_GR";
    ShipmentCarrier["DEALERSEND"] = "DEALERSEND";
    ShipmentCarrier["JOCOM"] = "JOCOM";
    ShipmentCarrier["CSE"] = "CSE";
    ShipmentCarrier["TFORCEFINALMILE"] = "TFORCE_FINALMILE";
    ShipmentCarrier["SHIPGATE"] = "SHIP_GATE";
    ShipmentCarrier["SHIPTER"] = "SHIPTER";
    ShipmentCarrier["NATIONALSAMEDAY"] = "NATIONAL_SAMEDAY";
    ShipmentCarrier["YUNEXPRESS"] = "YUNEXPRESS";
    ShipmentCarrier["CAINIAO"] = "CAINIAO";
    ShipmentCarrier["DMSMATRIX"] = "DMS_MATRIX";
    ShipmentCarrier["DIRECTLOG"] = "DIRECTLOG";
    ShipmentCarrier["ASENDIAUS"] = "ASENDIA_US";
    ShipmentCarrier["_3JMSLOGISTICS"] = "_3JMSLOGISTICS";
    ShipmentCarrier["LICCARDIEXPRESS"] = "LICCARDI_EXPRESS";
    ShipmentCarrier["SKYPOSTAL"] = "SKY_POSTAL";
    ShipmentCarrier["CNWANGTONG"] = "CNWANGTONG";
    ShipmentCarrier["POSTNORDLOGISTICSDK"] = "POSTNORD_LOGISTICS_DK";
    ShipmentCarrier["LOGISTIKA"] = "LOGISTIKA";
    ShipmentCarrier["CELERITAS"] = "CELERITAS";
    ShipmentCarrier["PRESSIODE"] = "PRESSIODE";
    ShipmentCarrier["SHREEMARUTI"] = "SHREE_MARUTI";
    ShipmentCarrier["LOGISTICSWORLDWIDEHK"] = "LOGISTICSWORLDWIDE_HK";
    ShipmentCarrier["EFEX"] = "EFEX";
    ShipmentCarrier["LOTTE"] = "LOTTE";
    ShipmentCarrier["LONESTAR"] = "LONESTAR";
    ShipmentCarrier["APRISAEXPRESS"] = "APRISAEXPRESS";
    ShipmentCarrier["BELRS"] = "BEL_RS";
    ShipmentCarrier["OSMWORLDWIDE"] = "OSM_WORLDWIDE";
    ShipmentCarrier["WESTGATEGL"] = "WESTGATE_GL";
    ShipmentCarrier["FASTRACK"] = "FASTRACK";
    ShipmentCarrier["DTDEXPR"] = "DTD_EXPR";
    ShipmentCarrier["ALFATREX"] = "ALFATREX";
    ShipmentCarrier["PROMEDDELIVERY"] = "PROMEDDELIVERY";
    ShipmentCarrier["THABITLOGISTICS"] = "THABIT_LOGISTICS";
    ShipmentCarrier["HCTLOGISTICS"] = "HCT_LOGISTICS";
    ShipmentCarrier["CARRYFLAP"] = "CARRY_FLAP";
    ShipmentCarrier["USOLDDOMINION"] = "US_OLD_DOMINION";
    ShipmentCarrier["ANICAMBOX"] = "ANICAM_BOX";
    ShipmentCarrier["WANBEXPRESS"] = "WANBEXPRESS";
    ShipmentCarrier["ANPOST"] = "AN_POST";
    ShipmentCarrier["DPDLOCAL"] = "DPD_LOCAL";
    ShipmentCarrier["STALLIONEXPRESS"] = "STALLIONEXPRESS";
    ShipmentCarrier["RAIDEREX"] = "RAIDEREX";
    ShipmentCarrier["SHOPFANS"] = "SHOPFANS";
    ShipmentCarrier["KYUNGDONGPARCEL"] = "KYUNGDONG_PARCEL";
    ShipmentCarrier["CHAMPIONLOGISTICS"] = "CHAMPION_LOGISTICS";
    ShipmentCarrier["PICKUPPSGP"] = "PICKUPP_SGP";
    ShipmentCarrier["MORNINGEXPRESS"] = "MORNING_EXPRESS";
    ShipmentCarrier["NACEX"] = "NACEX";
    ShipmentCarrier["THENILEWEBHOOK"] = "THENILE_WEBHOOK";
    ShipmentCarrier["HOLISOL"] = "HOLISOL";
    ShipmentCarrier["LBCEXPRESSFTP"] = "LBCEXPRESS_FTP";
    ShipmentCarrier["KURASI"] = "KURASI";
    ShipmentCarrier["USFREDDAWAY"] = "USF_REDDAWAY";
    ShipmentCarrier["APG"] = "APG";
    ShipmentCarrier["CNBOXC"] = "CN_BOXC";
    ShipmentCarrier["ECOSCOOTING"] = "ECOSCOOTING";
    ShipmentCarrier["MAINWAY"] = "MAINWAY";
    ShipmentCarrier["PAPERFLY"] = "PAPERFLY";
    ShipmentCarrier["HOUNDEXPRESS"] = "HOUNDEXPRESS";
    ShipmentCarrier["BOXBERRY"] = "BOX_BERRY";
    ShipmentCarrier["EPBOX"] = "EP_BOX";
    ShipmentCarrier["PLUSLOGUK"] = "PLUS_LOG_UK";
    ShipmentCarrier["FULFILLA"] = "FULFILLA";
    ShipmentCarrier["ASE"] = "ASE";
    ShipmentCarrier["MAILPLUS"] = "MAIL_PLUS";
    ShipmentCarrier["XPOLOGISTICS"] = "XPO_LOGISTICS";
    ShipmentCarrier["WNDIRECT"] = "WNDIRECT";
    ShipmentCarrier["CLOUDWISHASIA"] = "CLOUDWISH_ASIA";
    ShipmentCarrier["ZELERIS"] = "ZELERIS";
    ShipmentCarrier["GIOEXPRESS"] = "GIO_EXPRESS";
    ShipmentCarrier["OCSWORLDWIDE"] = "OCS_WORLDWIDE";
    ShipmentCarrier["ARKLOGISTICS"] = "ARK_LOGISTICS";
    ShipmentCarrier["AQUILINE"] = "AQUILINE";
    ShipmentCarrier["PILOTFREIGHT"] = "PILOT_FREIGHT";
    ShipmentCarrier["QWINTRY"] = "QWINTRY";
    ShipmentCarrier["DANSKEFRAGT"] = "DANSKE_FRAGT";
    ShipmentCarrier["CARRIERS"] = "CARRIERS";
    ShipmentCarrier["AIRCANADAGLOBAL"] = "AIR_CANADA_GLOBAL";
    ShipmentCarrier["PRESIDENTTRANS"] = "PRESIDENT_TRANS";
    ShipmentCarrier["STEPFORWARDFS"] = "STEPFORWARDFS";
    ShipmentCarrier["SKYNETUK"] = "SKYNET_UK";
    ShipmentCarrier["PITTOHIO"] = "PITTOHIO";
    ShipmentCarrier["CORREOSEXPRESS"] = "CORREOS_EXPRESS";
    ShipmentCarrier["RLUS"] = "RL_US";
    ShipmentCarrier["MARAXPRESS"] = "MARA_XPRESS";
    ShipmentCarrier["DESTINY"] = "DESTINY";
    ShipmentCarrier["UKYODEL"] = "UK_YODEL";
    ShipmentCarrier["COMETTECH"] = "COMET_TECH";
    ShipmentCarrier["DHLPARCELRU"] = "DHL_PARCEL_RU";
    ShipmentCarrier["TNTREFR"] = "TNT_REFR";
    ShipmentCarrier["SHREEANJANICOURIER"] = "SHREE_ANJANI_COURIER";
    ShipmentCarrier["MIKROPAKKETBE"] = "MIKROPAKKET_BE";
    ShipmentCarrier["ETSEXPRESS"] = "ETS_EXPRESS";
    ShipmentCarrier["COLISPRIVE"] = "COLIS_PRIVE";
    ShipmentCarrier["CNYUNDA"] = "CN_YUNDA";
    ShipmentCarrier["AAACOOPER"] = "AAA_COOPER";
    ShipmentCarrier["ROCKETPARCEL"] = "ROCKET_PARCEL";
    ShipmentCarrier["_360LION"] = "_360LION";
    ShipmentCarrier["PANDU"] = "PANDU";
    ShipmentCarrier["PROFESSIONALCOURIERS"] = "PROFESSIONAL_COURIERS";
    ShipmentCarrier["FLYTEXPRESS"] = "FLYTEXPRESS";
    ShipmentCarrier["LOGISTICSWORLDWIDEMY"] = "LOGISTICSWORLDWIDE_MY";
    ShipmentCarrier["CORREOSDEESPANA"] = "CORREOS_DE_ESPANA";
    ShipmentCarrier["IMX"] = "IMX";
    ShipmentCarrier["FOURPXEXPRESS"] = "FOUR_PX_EXPRESS";
    ShipmentCarrier["XPRESSBEES"] = "XPRESSBEES";
    ShipmentCarrier["PICKUPPVNM"] = "PICKUPP_VNM";
    ShipmentCarrier["STARTRACKEXPRESS1"] = "STARTRACK_EXPRESS";
    ShipmentCarrier["FRCOLISSIMO"] = "FR_COLISSIMO";
    ShipmentCarrier["NACEXSPAINREFERENCE"] = "NACEX_SPAIN_REFERENCE";
    ShipmentCarrier["DHLSUPPLYCHAINAU"] = "DHL_SUPPLY_CHAIN_AU";
    ShipmentCarrier["ESHIPPING"] = "ESHIPPING";
    ShipmentCarrier["SHREETIRUPATI"] = "SHREETIRUPATI";
    ShipmentCarrier["HXEXPRESS"] = "HX_EXPRESS";
    ShipmentCarrier["INDOPAKET"] = "INDOPAKET";
    ShipmentCarrier["CN17POST"] = "CN_17POST";
    ShipmentCarrier["K1EXPRESS"] = "K1_EXPRESS";
    ShipmentCarrier["CJGLS"] = "CJ_GLS";
    ShipmentCarrier["MYSGDEX"] = "MYS_GDEX";
    ShipmentCarrier["NATIONEX"] = "NATIONEX";
    ShipmentCarrier["ANJUN"] = "ANJUN";
    ShipmentCarrier["FARGOOD"] = "FARGOOD";
    ShipmentCarrier["SMGEXPRESS"] = "SMG_EXPRESS";
    ShipmentCarrier["RZYEXPRESS"] = "RZYEXPRESS";
    ShipmentCarrier["SEFL"] = "SEFL";
    ShipmentCarrier["TNTCLICKIT"] = "TNT_CLICK_IT";
    ShipmentCarrier["HDB"] = "HDB";
    ShipmentCarrier["HIPSHIPPER"] = "HIPSHIPPER";
    ShipmentCarrier["RPXLOGISTICS"] = "RPXLOGISTICS";
    ShipmentCarrier["KUEHNE"] = "KUEHNE";
    ShipmentCarrier["ITNEXIVE"] = "IT_NEXIVE";
    ShipmentCarrier["PTS"] = "PTS";
    ShipmentCarrier["SWISSPOSTFTP"] = "SWISS_POST_FTP";
    ShipmentCarrier["FASTRKSERV"] = "FASTRK_SERV";
    ShipmentCarrier["_472"] = "_4_72";
    ShipmentCarrier["USYRC"] = "US_YRC";
    ShipmentCarrier["POSTNLINTL3S"] = "POSTNL_INTL_3S";
    ShipmentCarrier["ELIANPOST"] = "ELIAN_POST";
    ShipmentCarrier["CUBYN"] = "CUBYN";
    ShipmentCarrier["SAUSAUDIPOST"] = "SAU_SAUDI_POST";
    ShipmentCarrier["ABXEXPRESSMY"] = "ABXEXPRESS_MY";
    ShipmentCarrier["HUAHANEXPRESS"] = "HUAHAN_EXPRESS";
    ShipmentCarrier["INDJAYONEXPRESS"] = "IND_JAYONEXPRESS";
    ShipmentCarrier["ZESEXPRESS"] = "ZES_EXPRESS";
    ShipmentCarrier["ZEPTOEXPRESS"] = "ZEPTO_EXPRESS";
    ShipmentCarrier["SKYNETZA"] = "SKYNET_ZA";
    ShipmentCarrier["ZEEK2DOOR"] = "ZEEK_2_DOOR";
    ShipmentCarrier["BLINKLASTMILE"] = "BLINKLASTMILE";
    ShipmentCarrier["POSTAUKR"] = "POSTA_UKR";
    ShipmentCarrier["CHROBINSON"] = "CHROBINSON";
    ShipmentCarrier["CNPOST56"] = "CN_POST56";
    ShipmentCarrier["COURANTPLUS"] = "COURANT_PLUS";
    ShipmentCarrier["SCUDEXEXPRESS"] = "SCUDEX_EXPRESS";
    ShipmentCarrier["SHIPENTEGRA"] = "SHIPENTEGRA";
    ShipmentCarrier["BTWOCEUROPE"] = "B_TWO_C_EUROPE";
    ShipmentCarrier["COPE"] = "COPE";
    ShipmentCarrier["INDGATI"] = "IND_GATI";
    ShipmentCarrier["CNWISHPOST"] = "CN_WISHPOST";
    ShipmentCarrier["NACEXES"] = "NACEX_ES";
    ShipmentCarrier["TAQBINHK"] = "TAQBIN_HK";
    ShipmentCarrier["GLOBALTRANZ"] = "GLOBALTRANZ";
    ShipmentCarrier["HKD"] = "HKD";
    ShipmentCarrier["BJSHOMEDELIVERY"] = "BJSHOMEDELIVERY";
    ShipmentCarrier["OMNIVA"] = "OMNIVA";
    ShipmentCarrier["SUTTON"] = "SUTTON";
    ShipmentCarrier["PANTHERREFERENCE"] = "PANTHER_REFERENCE";
    ShipmentCarrier["SFCSERVICE"] = "SFCSERVICE";
    ShipmentCarrier["LTL"] = "LTL";
    ShipmentCarrier["PARKNPARCEL"] = "PARKNPARCEL";
    ShipmentCarrier["SPRINGGDS"] = "SPRING_GDS";
    ShipmentCarrier["ECEXPRESS"] = "ECEXPRESS";
    ShipmentCarrier["INTERPARCELAU"] = "INTERPARCEL_AU";
    ShipmentCarrier["AGILITY"] = "AGILITY";
    ShipmentCarrier["XLEXPRESS"] = "XL_EXPRESS";
    ShipmentCarrier["ADERONLINE"] = "ADERONLINE";
    ShipmentCarrier["DIRECTCOURIERS"] = "DIRECTCOURIERS";
    ShipmentCarrier["PLANZER"] = "PLANZER";
    ShipmentCarrier["SENDING"] = "SENDING";
    ShipmentCarrier["NINJAVANWB"] = "NINJAVAN_WB";
    ShipmentCarrier["NATIONWIDEMY"] = "NATIONWIDE_MY";
    ShipmentCarrier["SENDIT"] = "SENDIT";
    ShipmentCarrier["GBARROW"] = "GB_ARROW";
    ShipmentCarrier["INDGOJAVAS"] = "IND_GOJAVAS";
    ShipmentCarrier["KPOST"] = "KPOST";
    ShipmentCarrier["DHLFREIGHT"] = "DHL_FREIGHT";
    ShipmentCarrier["BLUECARE"] = "BLUECARE";
    ShipmentCarrier["JINDOUYUN"] = "JINDOUYUN";
    ShipmentCarrier["TRACKON"] = "TRACKON";
    ShipmentCarrier["GBTUFFNELLS"] = "GB_TUFFNELLS";
    ShipmentCarrier["TRUMPCARD"] = "TRUMPCARD";
    ShipmentCarrier["ETOTAL"] = "ETOTAL";
    ShipmentCarrier["SFPLUSWEBHOOK"] = "SFPLUS_WEBHOOK";
    ShipmentCarrier["SEKOLOGISTICS"] = "SEKOLOGISTICS";
    ShipmentCarrier["HERMES2MANNHANDLING"] = "HERMES_2MANN_HANDLING";
    ShipmentCarrier["DPDLOCALREF"] = "DPD_LOCAL_REF";
    ShipmentCarrier["UDS"] = "UDS";
    ShipmentCarrier["ZASPECIALISEDFREIGHT"] = "ZA_SPECIALISED_FREIGHT";
    ShipmentCarrier["THAKERRY"] = "THA_KERRY";
    ShipmentCarrier["PRTINTSEUR"] = "PRT_INT_SEUR";
    ShipmentCarrier["BRACORREIOS"] = "BRA_CORREIOS";
    ShipmentCarrier["NZNZPOST"] = "NZ_NZ_POST";
    ShipmentCarrier["CNEQUICK"] = "CN_EQUICK";
    ShipmentCarrier["MYSEMS"] = "MYS_EMS";
    ShipmentCarrier["GBNORSK"] = "GB_NORSK";
    ShipmentCarrier["ESPMRW"] = "ESP_MRW";
    ShipmentCarrier["ESPPACKLINK"] = "ESP_PACKLINK";
    ShipmentCarrier["KANGAROOMY"] = "KANGAROO_MY";
    ShipmentCarrier["RPX"] = "RPX";
    ShipmentCarrier["XDPUKREFERENCE"] = "XDP_UK_REFERENCE";
    ShipmentCarrier["NINJAVANMY"] = "NINJAVAN_MY";
    ShipmentCarrier["ADICIONAL"] = "ADICIONAL";
    ShipmentCarrier["NINJAVANID"] = "NINJAVAN_ID";
    ShipmentCarrier["ROADBULL"] = "ROADBULL";
    ShipmentCarrier["YAKIT"] = "YAKIT";
    ShipmentCarrier["MAILAMERICAS"] = "MAILAMERICAS";
    ShipmentCarrier["MIKROPAKKET"] = "MIKROPAKKET";
    ShipmentCarrier["DYNALOGIC"] = "DYNALOGIC";
    ShipmentCarrier["DHLES"] = "DHL_ES";
    ShipmentCarrier["DHLPARCELNL"] = "DHL_PARCEL_NL";
    ShipmentCarrier["DHLGLOBALMAILASIA"] = "DHL_GLOBAL_MAIL_ASIA";
    ShipmentCarrier["DAWNWING"] = "DAWN_WING";
    ShipmentCarrier["GENIKIGR"] = "GENIKI_GR";
    ShipmentCarrier["HERMESWORLDUK"] = "HERMESWORLD_UK";
    ShipmentCarrier["ALPHAFAST"] = "ALPHAFAST";
    ShipmentCarrier["BUYLOGIC"] = "BUYLOGIC";
    ShipmentCarrier["EKART"] = "EKART";
    ShipmentCarrier["MEXSENDA"] = "MEX_SENDA";
    ShipmentCarrier["SFCLOGISTICS"] = "SFC_LOGISTICS";
    ShipmentCarrier["POSTSERBIA"] = "POST_SERBIA";
    ShipmentCarrier["INDDELHIVERY"] = "IND_DELHIVERY";
    ShipmentCarrier["DEDPDDELISTRACK"] = "DE_DPD_DELISTRACK";
    ShipmentCarrier["RPD2MAN"] = "RPD2MAN";
    ShipmentCarrier["CNSFEXPRESS"] = "CN_SF_EXPRESS";
    ShipmentCarrier["YANWEN"] = "YANWEN";
    ShipmentCarrier["MYSSKYNET"] = "MYS_SKYNET";
    ShipmentCarrier["CORREOSDEMEXICO"] = "CORREOS_DE_MEXICO";
    ShipmentCarrier["CBLLOGISTICA"] = "CBL_LOGISTICA";
    ShipmentCarrier["MEXESTAFETA"] = "MEX_ESTAFETA";
    ShipmentCarrier["AUAUSTRIANPOST"] = "AU_AUSTRIAN_POST";
    ShipmentCarrier["RINCOS"] = "RINCOS";
    ShipmentCarrier["NLDDHL"] = "NLD_DHL";
    ShipmentCarrier["RUSSIANPOST"] = "RUSSIAN_POST";
    ShipmentCarrier["COURIERSPLEASE"] = "COURIERS_PLEASE";
    ShipmentCarrier["POSTNORDLOGISTICS"] = "POSTNORD_LOGISTICS";
    ShipmentCarrier["FEDEX"] = "FEDEX";
    ShipmentCarrier["DPEEXPRESS"] = "DPE_EXPRESS";
    ShipmentCarrier["DPD"] = "DPD";
    ShipmentCarrier["ADSONE"] = "ADSONE";
    ShipmentCarrier["IDNJNE"] = "IDN_JNE";
    ShipmentCarrier["THECOURIERGUY"] = "THECOURIERGUY";
    ShipmentCarrier["CNEXPS"] = "CNEXPS";
    ShipmentCarrier["PRTCHRONOPOST"] = "PRT_CHRONOPOST";
    ShipmentCarrier["LANDMARKGLOBAL"] = "LANDMARK_GLOBAL";
    ShipmentCarrier["ITDHLECOMMERCE"] = "IT_DHL_ECOMMERCE";
    ShipmentCarrier["ESPNACEX"] = "ESP_NACEX";
    ShipmentCarrier["PRTCTT"] = "PRT_CTT";
    ShipmentCarrier["BEKIALA"] = "BE_KIALA";
    ShipmentCarrier["ASENDIAUK"] = "ASENDIA_UK";
    ShipmentCarrier["GLOBALTNT"] = "GLOBAL_TNT";
    ShipmentCarrier["POSTURIS"] = "POSTUR_IS";
    ShipmentCarrier["EPARCELKR"] = "EPARCEL_KR";
    ShipmentCarrier["INPOSTPACZKOMATY"] = "INPOST_PACZKOMATY";
    ShipmentCarrier["ITPOSTEITALIA"] = "IT_POSTE_ITALIA";
    ShipmentCarrier["BEBPOST"] = "BE_BPOST";
    ShipmentCarrier["PLPOCZTAPOLSKA"] = "PL_POCZTA_POLSKA";
    ShipmentCarrier["MYSMYSPOST"] = "MYS_MYS_POST";
    ShipmentCarrier["SGSGPOST"] = "SG_SG_POST";
    ShipmentCarrier["THATHAILANDPOST"] = "THA_THAILAND_POST";
    ShipmentCarrier["LEXSHIP"] = "LEXSHIP";
    ShipmentCarrier["FASTWAYNZ"] = "FASTWAY_NZ";
    ShipmentCarrier["DHLAU"] = "DHL_AU";
    ShipmentCarrier["COSTMETICSNOW"] = "COSTMETICSNOW";
    ShipmentCarrier["PFLOGISTICS"] = "PFLOGISTICS";
    ShipmentCarrier["LOOMISEXPRESS"] = "LOOMIS_EXPRESS";
    ShipmentCarrier["GLSITALY"] = "GLS_ITALY";
    ShipmentCarrier["LINE"] = "LINE";
    ShipmentCarrier["GELEXPRESS"] = "GEL_EXPRESS";
    ShipmentCarrier["HUODULL"] = "HUODULL";
    ShipmentCarrier["NINJAVANSG"] = "NINJAVAN_SG";
    ShipmentCarrier["JANIO"] = "JANIO";
    ShipmentCarrier["AOCOURIER"] = "AO_COURIER";
    ShipmentCarrier["BRTITSENDERREF"] = "BRT_IT_SENDER_REF";
    ShipmentCarrier["SAILPOST"] = "SAILPOST";
    ShipmentCarrier["LALAMOVE"] = "LALAMOVE";
    ShipmentCarrier["NEWZEALANDCOURIERS"] = "NEWZEALAND_COURIERS";
    ShipmentCarrier["ETOMARS"] = "ETOMARS";
    ShipmentCarrier["VIRTRANSPORT"] = "VIRTRANSPORT";
    ShipmentCarrier["WIZMO"] = "WIZMO";
    ShipmentCarrier["PALLETWAYS"] = "PALLETWAYS";
    ShipmentCarrier["IDIKA"] = "I_DIKA";
    ShipmentCarrier["CFLLOGISTICS"] = "CFL_LOGISTICS";
    ShipmentCarrier["GEMWORLDWIDE"] = "GEMWORLDWIDE";
    ShipmentCarrier["GLOBALEXPRESS"] = "GLOBAL_EXPRESS";
    ShipmentCarrier["LOGISTYXTRANSGROUP"] = "LOGISTYX_TRANSGROUP";
    ShipmentCarrier["WESTBANKCOURIER"] = "WESTBANK_COURIER";
    ShipmentCarrier["ARCOSPEDIZIONI"] = "ARCO_SPEDIZIONI";
    ShipmentCarrier["YDHEXPRESS"] = "YDH_EXPRESS";
    ShipmentCarrier["PARCELINKLOGISTICS"] = "PARCELINKLOGISTICS";
    ShipmentCarrier["CNDEXPRESS"] = "CNDEXPRESS";
    ShipmentCarrier["NOXNIGHTTIMEEXPRESS"] = "NOX_NIGHT_TIME_EXPRESS";
    ShipmentCarrier["AERONET"] = "AERONET";
    ShipmentCarrier["LTIANEXP"] = "LTIANEXP";
    ShipmentCarrier["INTEGRA2FTP"] = "INTEGRA2_FTP";
    ShipmentCarrier["PARCELONE"] = "PARCELONE";
    ShipmentCarrier["NOXNACHTEXPRESS"] = "NOX_NACHTEXPRESS";
    ShipmentCarrier["CNCHINAPOSTEMS"] = "CN_CHINA_POST_EMS";
    ShipmentCarrier["CHUKOU1"] = "CHUKOU1";
    ShipmentCarrier["GLSSLOV"] = "GLS_SLOV";
    ShipmentCarrier["ORANGEDS"] = "ORANGE_DS";
    ShipmentCarrier["JOOMLOGIS"] = "JOOM_LOGIS";
    ShipmentCarrier["AUSSTARTRACK"] = "AUS_STARTRACK";
    ShipmentCarrier["DHL"] = "DHL";
    ShipmentCarrier["GBAPC"] = "GB_APC";
    ShipmentCarrier["BONDSCOURIERS"] = "BONDSCOURIERS";
    ShipmentCarrier["JPNJAPANPOST"] = "JPN_JAPAN_POST";
    ShipmentCarrier["USPS"] = "USPS";
    ShipmentCarrier["WINIT"] = "WINIT";
    ShipmentCarrier["ARGOCA"] = "ARG_OCA";
    ShipmentCarrier["TWTAIWANPOST"] = "TW_TAIWAN_POST";
    ShipmentCarrier["DMMNETWORK"] = "DMM_NETWORK";
    ShipmentCarrier["TNT"] = "TNT";
    ShipmentCarrier["BHPOSTA"] = "BH_POSTA";
    ShipmentCarrier["SWEPOSTNORD"] = "SWE_POSTNORD";
    ShipmentCarrier["CACANADAPOST"] = "CA_CANADA_POST";
    ShipmentCarrier["WISELOADS"] = "WISELOADS";
    ShipmentCarrier["ASENDIAHK"] = "ASENDIA_HK";
    ShipmentCarrier["NLDGLS"] = "NLD_GLS";
    ShipmentCarrier["MEXREDPACK"] = "MEX_REDPACK";
    ShipmentCarrier["JETSHIP"] = "JET_SHIP";
    ShipmentCarrier["DEDHLEXPRESS"] = "DE_DHL_EXPRESS";
    ShipmentCarrier["NINJAVANTHAI"] = "NINJAVAN_THAI";
    ShipmentCarrier["RABENGROUP"] = "RABEN_GROUP";
    ShipmentCarrier["ESPASM"] = "ESP_ASM";
    ShipmentCarrier["HRVHRVATSKA"] = "HRV_HRVATSKA";
    ShipmentCarrier["GLOBALESTES"] = "GLOBAL_ESTES";
    ShipmentCarrier["LTULIETUVOS"] = "LTU_LIETUVOS";
    ShipmentCarrier["BELDHL"] = "BEL_DHL";
    ShipmentCarrier["AUAUPOST"] = "AU_AU_POST";
    ShipmentCarrier["SPEEDEXCOURIER"] = "SPEEDEXCOURIER";
    ShipmentCarrier["FRCOLIS"] = "FR_COLIS";
    ShipmentCarrier["ARAMEX"] = "ARAMEX";
    ShipmentCarrier["DPEX"] = "DPEX";
    ShipmentCarrier["MYSAIRPAK"] = "MYS_AIRPAK";
    ShipmentCarrier["CUCKOOEXPRESS"] = "CUCKOOEXPRESS";
    ShipmentCarrier["DPDPOLAND"] = "DPD_POLAND";
    ShipmentCarrier["NLDPOSTNL"] = "NLD_POSTNL";
    ShipmentCarrier["NIMEXPRESS"] = "NIM_EXPRESS";
    ShipmentCarrier["QUANTIUM"] = "QUANTIUM";
    ShipmentCarrier["SENDLE"] = "SENDLE";
    ShipmentCarrier["ESPREDUR"] = "ESP_REDUR";
    ShipmentCarrier["MATKAHUOLTO"] = "MATKAHUOLTO";
    ShipmentCarrier["CPACKET"] = "CPACKET";
    ShipmentCarrier["POSTI"] = "POSTI";
    ShipmentCarrier["HUNTEREXPRESS"] = "HUNTER_EXPRESS";
    ShipmentCarrier["CHOIREXP"] = "CHOIR_EXP";
    ShipmentCarrier["LEGIONEXPRESS"] = "LEGION_EXPRESS";
    ShipmentCarrier["AUSTRIANPOSTEXPRESS"] = "AUSTRIAN_POST_EXPRESS";
    ShipmentCarrier["GRUPO"] = "GRUPO";
    ShipmentCarrier["POSTARO"] = "POSTA_RO";
    ShipmentCarrier["INTERPARCELUK"] = "INTERPARCEL_UK";
    ShipmentCarrier["GLOBALABF"] = "GLOBAL_ABF";
    ShipmentCarrier["POSTENNORGE"] = "POSTEN_NORGE";
    ShipmentCarrier["XPERTDELIVERY"] = "XPERT_DELIVERY";
    ShipmentCarrier["DHLREFR"] = "DHL_REFR";
    ShipmentCarrier["DHLHK"] = "DHL_HK";
    ShipmentCarrier["SKYNETUAE"] = "SKYNET_UAE";
    ShipmentCarrier["GOJEK"] = "GOJEK";
    ShipmentCarrier["YODELINTNL"] = "YODEL_INTNL";
    ShipmentCarrier["JANCO"] = "JANCO";
    ShipmentCarrier["YTO"] = "YTO";
    ShipmentCarrier["WISEEXPRESS"] = "WISE_EXPRESS";
    ShipmentCarrier["JTEXPRESSVN"] = "JTEXPRESS_VN";
    ShipmentCarrier["FEDEXINTLMLSERV"] = "FEDEX_INTL_MLSERV";
    ShipmentCarrier["VAMOX"] = "VAMOX";
    ShipmentCarrier["AMSGRP"] = "AMS_GRP";
    ShipmentCarrier["DHLJP"] = "DHL_JP";
    ShipmentCarrier["HRPARCEL"] = "HRPARCEL";
    ShipmentCarrier["GESWL"] = "GESWL";
    ShipmentCarrier["BLUESTAR"] = "BLUESTAR";
    ShipmentCarrier["CDEKTR"] = "CDEK_TR";
    ShipmentCarrier["DESCARTES"] = "DESCARTES";
    ShipmentCarrier["DELTECUK"] = "DELTEC_UK";
    ShipmentCarrier["DTDCEXPRESS"] = "DTDC_EXPRESS";
    ShipmentCarrier["TOURLINE"] = "TOURLINE";
    ShipmentCarrier["BHWORLDWIDE"] = "BH_WORLDWIDE";
    ShipmentCarrier["OCS"] = "OCS";
    ShipmentCarrier["YINGNUOLOGISTICS"] = "YINGNUO_LOGISTICS";
    ShipmentCarrier["UPS"] = "UPS";
    ShipmentCarrier["TOLL"] = "TOLL";
    ShipmentCarrier["PRTSEUR"] = "PRT_SEUR";
    ShipmentCarrier["DTDCAU"] = "DTDC_AU";
    ShipmentCarrier["THADYNAMICLOGISTICS"] = "THA_DYNAMIC_LOGISTICS";
    ShipmentCarrier["UBILOGISTICS"] = "UBI_LOGISTICS";
    ShipmentCarrier["FEDEXCROSSBORDER"] = "FEDEX_CROSSBORDER";
    ShipmentCarrier["A1POST"] = "A1POST";
    ShipmentCarrier["TAZMANIANFREIGHT"] = "TAZMANIAN_FREIGHT";
    ShipmentCarrier["CJINTMY"] = "CJ_INT_MY";
    ShipmentCarrier["SAIAFREIGHT"] = "SAIA_FREIGHT";
    ShipmentCarrier["SGQXPRESS"] = "SG_QXPRESS";
    ShipmentCarrier["NHANSSOLUTIONS"] = "NHANS_SOLUTIONS";
    ShipmentCarrier["DPDFR"] = "DPD_FR";
    ShipmentCarrier["COORDINADORA"] = "COORDINADORA";
    ShipmentCarrier["ANDREANI"] = "ANDREANI";
    ShipmentCarrier["DOORA"] = "DOORA";
    ShipmentCarrier["INTERPARCELNZ"] = "INTERPARCEL_NZ";
    ShipmentCarrier["PHLJAMEXPRESS"] = "PHL_JAMEXPRESS";
    ShipmentCarrier["BELBELGIUMPOST"] = "BEL_BELGIUM_POST";
    ShipmentCarrier["USAPC"] = "US_APC";
    ShipmentCarrier["IDNPOS"] = "IDN_POS";
    ShipmentCarrier["FRMONDIAL"] = "FR_MONDIAL";
    ShipmentCarrier["DEDHL"] = "DE_DHL";
    ShipmentCarrier["HKRPX"] = "HK_RPX";
    ShipmentCarrier["DHLPIECEID"] = "DHL_PIECEID";
    ShipmentCarrier["VNPOSTEMS"] = "VNPOST_EMS";
    ShipmentCarrier["RRDONNELLEY"] = "RRDONNELLEY";
    ShipmentCarrier["DPDDE"] = "DPD_DE";
    ShipmentCarrier["DELCARTIN"] = "DELCART_IN";
    ShipmentCarrier["IMEXGLOBALSOLUTIONS"] = "IMEXGLOBALSOLUTIONS";
    ShipmentCarrier["ACOMMERCE"] = "ACOMMERCE";
    ShipmentCarrier["EURODIS"] = "EURODIS";
    ShipmentCarrier["CANPAR"] = "CANPAR";
    ShipmentCarrier["GLS"] = "GLS";
    ShipmentCarrier["INDECOM"] = "IND_ECOM";
    ShipmentCarrier["ESPENVIALIA"] = "ESP_ENVIALIA";
    ShipmentCarrier["DHLUK"] = "DHL_UK";
    ShipmentCarrier["SMSAEXPRESS"] = "SMSA_EXPRESS";
    ShipmentCarrier["TNTFR"] = "TNT_FR";
    ShipmentCarrier["DEXI"] = "DEX_I";
    ShipmentCarrier["BUDBEEWEBHOOK"] = "BUDBEE_WEBHOOK";
    ShipmentCarrier["COPACOURIER"] = "COPA_COURIER";
    ShipmentCarrier["VNMVIETNAMPOST"] = "VNM_VIETNAM_POST";
    ShipmentCarrier["DPDHK"] = "DPD_HK";
    ShipmentCarrier["TOLLNZ"] = "TOLL_NZ";
    ShipmentCarrier["ECHO"] = "ECHO";
    ShipmentCarrier["FEDEXFR"] = "FEDEX_FR";
    ShipmentCarrier["BORDEREXPRESS"] = "BORDEREXPRESS";
    ShipmentCarrier["MAILPLUSJPN"] = "MAILPLUS_JPN";
    ShipmentCarrier["TNTUKREFR"] = "TNT_UK_REFR";
    ShipmentCarrier["KEC"] = "KEC";
    ShipmentCarrier["DPDRO"] = "DPD_RO";
    ShipmentCarrier["TNTJP"] = "TNT_JP";
    ShipmentCarrier["THCJ"] = "TH_CJ";
    ShipmentCarrier["ECCN"] = "EC_CN";
    ShipmentCarrier["FASTWAYUK"] = "FASTWAY_UK";
    ShipmentCarrier["FASTWAYUS"] = "FASTWAY_US";
    ShipmentCarrier["GLSDE"] = "GLS_DE";
    ShipmentCarrier["GLSES"] = "GLS_ES";
    ShipmentCarrier["GLSFR"] = "GLS_FR";
    ShipmentCarrier["MONDIALBE"] = "MONDIAL_BE";
    ShipmentCarrier["SGTIT"] = "SGT_IT";
    ShipmentCarrier["TNTCN"] = "TNT_CN";
    ShipmentCarrier["TNTDE"] = "TNT_DE";
    ShipmentCarrier["TNTES"] = "TNT_ES";
    ShipmentCarrier["TNTPL"] = "TNT_PL";
    ShipmentCarrier["PARCELFORCE"] = "PARCELFORCE";
    ShipmentCarrier["SWISSPOST"] = "SWISS_POST";
    ShipmentCarrier["TOLLIPEC"] = "TOLL_IPEC";
    ShipmentCarrier["AIR21"] = "AIR_21";
    ShipmentCarrier["AIRSPEED"] = "AIRSPEED";
    ShipmentCarrier["BERT"] = "BERT";
    ShipmentCarrier["BLUEDART"] = "BLUEDART";
    ShipmentCarrier["COLLECTPLUS"] = "COLLECTPLUS";
    ShipmentCarrier["COURIERPLUS"] = "COURIERPLUS";
    ShipmentCarrier["COURIERPOST"] = "COURIER_POST";
    ShipmentCarrier["DHLGLOBALMAIL"] = "DHL_GLOBAL_MAIL";
    ShipmentCarrier["DPDUK"] = "DPD_UK";
    ShipmentCarrier["DELTECDE"] = "DELTEC_DE";
    ShipmentCarrier["DEUTSCHEDE"] = "DEUTSCHE_DE";
    ShipmentCarrier["DOTZOT"] = "DOTZOT";
    ShipmentCarrier["ELTAGR"] = "ELTA_GR";
    ShipmentCarrier["EMSCN"] = "EMS_CN";
    ShipmentCarrier["ECARGO"] = "ECARGO";
    ShipmentCarrier["ENSENDA"] = "ENSENDA";
    ShipmentCarrier["FERCAMIT"] = "FERCAM_IT";
    ShipmentCarrier["FASTWAYZA"] = "FASTWAY_ZA";
    ShipmentCarrier["FASTWAYAU"] = "FASTWAY_AU";
    ShipmentCarrier["FIRSTLOGISITCS"] = "FIRST_LOGISITCS";
    ShipmentCarrier["GEODIS"] = "GEODIS";
    ShipmentCarrier["GLOBEGISTICS"] = "GLOBEGISTICS";
    ShipmentCarrier["GREYHOUND"] = "GREYHOUND";
    ShipmentCarrier["JETSHIPMY"] = "JETSHIP_MY";
    ShipmentCarrier["LIONPARCEL"] = "LION_PARCEL";
    ShipmentCarrier["AEROFLASH"] = "AEROFLASH";
    ShipmentCarrier["ONTRAC"] = "ONTRAC";
    ShipmentCarrier["SAGAWA"] = "SAGAWA";
    ShipmentCarrier["SIODEMKA"] = "SIODEMKA";
    ShipmentCarrier["STARTRACK"] = "STARTRACK";
    ShipmentCarrier["TNTAU"] = "TNT_AU";
    ShipmentCarrier["TNTIT"] = "TNT_IT";
    ShipmentCarrier["TRANSMISSION"] = "TRANSMISSION";
    ShipmentCarrier["YAMATO"] = "YAMATO";
    ShipmentCarrier["DHLIT"] = "DHL_IT";
    ShipmentCarrier["DHLAT"] = "DHL_AT";
    ShipmentCarrier["LOGISTICSWORLDWIDEKR"] = "LOGISTICSWORLDWIDE_KR";
    ShipmentCarrier["GLSSPAIN"] = "GLS_SPAIN";
    ShipmentCarrier["AMAZONUKAPI"] = "AMAZON_UK_API";
    ShipmentCarrier["DPDFRREFERENCE"] = "DPD_FR_REFERENCE";
    ShipmentCarrier["DHLPARCELUK"] = "DHLPARCEL_UK";
    ShipmentCarrier["MEGASAVE"] = "MEGASAVE";
    ShipmentCarrier["QUALITYPOST"] = "QUALITYPOST";
    ShipmentCarrier["IDSLOGISTICS"] = "IDS_LOGISTICS";
    ShipmentCarrier["JOYINGBOX1"] = "JOYINGBOX";
    ShipmentCarrier["PANTHERORDERNUMBER"] = "PANTHER_ORDER_NUMBER";
    ShipmentCarrier["WATKINSSHEPARD"] = "WATKINS_SHEPARD";
    ShipmentCarrier["FASTTRACK"] = "FASTTRACK";
    ShipmentCarrier["UPEXPRESS"] = "UP_EXPRESS";
    ShipmentCarrier["ELOGISTICA"] = "ELOGISTICA";
    ShipmentCarrier["ECOURIER"] = "ECOURIER";
    ShipmentCarrier["CJPHILIPPINES"] = "CJ_PHILIPPINES";
    ShipmentCarrier["SPEEDEX"] = "SPEEDEX";
    ShipmentCarrier["ORANGECONNEX"] = "ORANGECONNEX";
    ShipmentCarrier["TECOR"] = "TECOR";
    ShipmentCarrier["SAEE"] = "SAEE";
    ShipmentCarrier["GLSITALYFTP"] = "GLS_ITALY_FTP";
    ShipmentCarrier["DELIVERE"] = "DELIVERE";
    ShipmentCarrier["YYCOM"] = "YYCOM";
    ShipmentCarrier["ADICIONALPT"] = "ADICIONAL_PT";
    ShipmentCarrier["DKSH"] = "DKSH";
    ShipmentCarrier["NIPPONEXPRESSFTP"] = "NIPPON_EXPRESS_FTP";
    ShipmentCarrier["GOLS"] = "GOLS";
    ShipmentCarrier["FUJEXP"] = "FUJEXP";
    ShipmentCarrier["QTRACK"] = "QTRACK";
    ShipmentCarrier["OMLOGISTICSAPI"] = "OMLOGISTICS_API";
    ShipmentCarrier["GDPHARM"] = "GDPHARM";
    ShipmentCarrier["MISUMICN"] = "MISUMI_CN";
    ShipmentCarrier["AIRCANADA"] = "AIR_CANADA";
    ShipmentCarrier["CITY56WEBHOOK"] = "CITY56_WEBHOOK";
    ShipmentCarrier["SAGAWAAPI"] = "SAGAWA_API";
    ShipmentCarrier["KEDAEX"] = "KEDAEX";
    ShipmentCarrier["PGEONAPI"] = "PGEON_API";
    ShipmentCarrier["WEWORLDEXPRESS"] = "WEWORLDEXPRESS";
    ShipmentCarrier["JTLOGISTICS"] = "JT_LOGISTICS";
    ShipmentCarrier["TRUSK"] = "TRUSK";
    ShipmentCarrier["VIAXPRESS"] = "VIAXPRESS";
    ShipmentCarrier["DHLSUPPLYCHAINID"] = "DHL_SUPPLYCHAIN_ID";
    ShipmentCarrier["ZUELLIGPHARMASFTP"] = "ZUELLIGPHARMA_SFTP";
    ShipmentCarrier["MEEST"] = "MEEST";
    ShipmentCarrier["TOLLPRIORITY"] = "TOLL_PRIORITY";
    ShipmentCarrier["MOTHERSHIPAPI"] = "MOTHERSHIP_API";
    ShipmentCarrier["CAPITAL"] = "CAPITAL";
    ShipmentCarrier["EUROPAKETAPI"] = "EUROPAKET_API";
    ShipmentCarrier["HFD"] = "HFD";
    ShipmentCarrier["TOURLINEREFERENCE"] = "TOURLINE_REFERENCE";
    ShipmentCarrier["GIOECOURIER"] = "GIO_ECOURIER";
    ShipmentCarrier["CNLOGISTICS"] = "CN_LOGISTICS";
    ShipmentCarrier["PANDION"] = "PANDION";
    ShipmentCarrier["BPOSTAPI"] = "BPOST_API";
    ShipmentCarrier["PASSPORTSHIPPING"] = "PASSPORTSHIPPING";
    ShipmentCarrier["PAKAJO"] = "PAKAJO";
    ShipmentCarrier["DACHSER"] = "DACHSER";
    ShipmentCarrier["YUSENSFTP"] = "YUSEN_SFTP";
    ShipmentCarrier["SHYPLITE"] = "SHYPLITE";
    ShipmentCarrier["XYY"] = "XYY";
    ShipmentCarrier["MWD"] = "MWD";
    ShipmentCarrier["FAXECARGO"] = "FAXECARGO";
    ShipmentCarrier["MAZET"] = "MAZET";
    ShipmentCarrier["FIRSTLOGISTICSAPI"] = "FIRST_LOGISTICS_API";
    ShipmentCarrier["SPRINTPACK"] = "SPRINT_PACK";
    ShipmentCarrier["HERMESDEFTP"] = "HERMES_DE_FTP";
    ShipmentCarrier["CONCISE"] = "CONCISE";
    ShipmentCarrier["KERRYEXPRESSTWAPI"] = "KERRY_EXPRESS_TW_API";
    ShipmentCarrier["EWE"] = "EWE";
    ShipmentCarrier["FASTDESPATCH"] = "FASTDESPATCH";
    ShipmentCarrier["ABCUSTOMSFTP"] = "ABCUSTOM_SFTP";
    ShipmentCarrier["CHAZKI"] = "CHAZKI";
    ShipmentCarrier["SHIPPIE"] = "SHIPPIE";
    ShipmentCarrier["GEODISAPI"] = "GEODIS_API";
    ShipmentCarrier["NAQELEXPRESS"] = "NAQEL_EXPRESS";
    ShipmentCarrier["PAPAWEBHOOK"] = "PAPA_WEBHOOK";
    ShipmentCarrier["FORWARDAIR"] = "FORWARDAIR";
    ShipmentCarrier["DIALOGOLOGISTICAAPI"] = "DIALOGO_LOGISTICA_API";
    ShipmentCarrier["LALAMOVEAPI"] = "LALAMOVE_API";
    ShipmentCarrier["TOMYDOOR"] = "TOMYDOOR";
    ShipmentCarrier["KRONOSWEBHOOK"] = "KRONOS_WEBHOOK";
    ShipmentCarrier["JTCARGO"] = "JTCARGO";
    ShipmentCarrier["TCAT"] = "T_CAT";
    ShipmentCarrier["CONCISEWEBHOOK"] = "CONCISE_WEBHOOK";
    ShipmentCarrier["TELEPORTWEBHOOK"] = "TELEPORT_WEBHOOK";
    ShipmentCarrier["CUSTOMCOAPI"] = "CUSTOMCO_API";
    ShipmentCarrier["SPXTH"] = "SPX_TH";
    ShipmentCarrier["BOLLORELOGISTICS"] = "BOLLORE_LOGISTICS";
    ShipmentCarrier["CLICKLINKSFTP"] = "CLICKLINK_SFTP";
    ShipmentCarrier["M3LOGISTICS"] = "M3LOGISTICS";
    ShipmentCarrier["VNPOSTAPI"] = "VNPOST_API";
    ShipmentCarrier["AXLEHIREFTP"] = "AXLEHIRE_FTP";
    ShipmentCarrier["SHADOWFAX"] = "SHADOWFAX";
    ShipmentCarrier["MYHERMESUKAPI"] = "MYHERMES_UK_API";
    ShipmentCarrier["DAIICHI"] = "DAIICHI";
    ShipmentCarrier["MENSAJEROSURBANOSAPI"] = "MENSAJEROSURBANOS_API";
    ShipmentCarrier["POLARSPEED"] = "POLARSPEED";
    ShipmentCarrier["IDEXPRESSID"] = "IDEXPRESS_ID";
    ShipmentCarrier["PAYO"] = "PAYO";
    ShipmentCarrier["WHISTLSFTP"] = "WHISTL_SFTP";
    ShipmentCarrier["INTEXDE"] = "INTEX_DE";
    ShipmentCarrier["TRANS2U"] = "TRANS2U";
    ShipmentCarrier["PRODUCTCAREGROUPSFTP"] = "PRODUCTCAREGROUP_SFTP";
    ShipmentCarrier["BIGSMART"] = "BIGSMART";
    ShipmentCarrier["EXPEDITORSAPIREF"] = "EXPEDITORS_API_REF";
    ShipmentCarrier["AITWORLDWIDEAPI"] = "AITWORLDWIDE_API";
    ShipmentCarrier["WORLDCOURIER"] = "WORLDCOURIER";
    ShipmentCarrier["QUIQUP"] = "QUIQUP";
    ShipmentCarrier["AGEDISSSFTP"] = "AGEDISS_SFTP";
    ShipmentCarrier["ANDREANIAPI"] = "ANDREANI_API";
    ShipmentCarrier["CRLEXPRESS"] = "CRLEXPRESS";
    ShipmentCarrier["SMARTCAT"] = "SMARTCAT";
    ShipmentCarrier["CROSSFLIGHT"] = "CROSSFLIGHT";
    ShipmentCarrier["PROCARRIER"] = "PROCARRIER";
    ShipmentCarrier["DHLREFERENCEAPI"] = "DHL_REFERENCE_API";
    ShipmentCarrier["SEINOAPI"] = "SEINO_API";
    ShipmentCarrier["WSPEXPRESS"] = "WSPEXPRESS";
    ShipmentCarrier["KRONOS"] = "KRONOS";
    ShipmentCarrier["TOTALEXPRESSAPI"] = "TOTAL_EXPRESS_API";
    ShipmentCarrier["PARCLL"] = "PARCLL";
    ShipmentCarrier["XPEDIGO"] = "XPEDIGO";
    ShipmentCarrier["STARTRACKWEBHOOK"] = "STAR_TRACK_WEBHOOK";
    ShipmentCarrier["GPOST"] = "GPOST";
    ShipmentCarrier["UCS"] = "UCS";
    ShipmentCarrier["DMFGROUP"] = "DMFGROUP";
    ShipmentCarrier["COORDINADORAAPI"] = "COORDINADORA_API";
    ShipmentCarrier["MARKEN"] = "MARKEN";
    ShipmentCarrier["NTL"] = "NTL";
    ShipmentCarrier["REDJEPAKKETJE"] = "REDJEPAKKETJE";
    ShipmentCarrier["ALLIEDEXPRESSFTP"] = "ALLIED_EXPRESS_FTP";
    ShipmentCarrier["MONDIALRELAYES"] = "MONDIALRELAY_ES";
    ShipmentCarrier["NAEKOFTP"] = "NAEKO_FTP";
    ShipmentCarrier["MHI"] = "MHI";
    ShipmentCarrier["SHIPPIFY"] = "SHIPPIFY";
    ShipmentCarrier["MALCAAMITAPI"] = "MALCA_AMIT_API";
    ShipmentCarrier["JTEXPRESSSGAPI"] = "JTEXPRESS_SG_API";
    ShipmentCarrier["DACHSERWEB"] = "DACHSER_WEB";
    ShipmentCarrier["FLIGHTLG"] = "FLIGHTLG";
    ShipmentCarrier["CAGO"] = "CAGO";
    ShipmentCarrier["COM1EXPRESS"] = "COM1EXPRESS";
    ShipmentCarrier["TONAMIFTP"] = "TONAMI_FTP";
    ShipmentCarrier["PACKFLEET"] = "PACKFLEET";
    ShipmentCarrier["PUROLATORINTERNATIONAL"] = "PUROLATOR_INTERNATIONAL";
    ShipmentCarrier["WINESHIPPINGWEBHOOK"] = "WINESHIPPING_WEBHOOK";
    ShipmentCarrier["DHLESSFTP"] = "DHL_ES_SFTP";
    ShipmentCarrier["PCHOMEAPI"] = "PCHOME_API";
    ShipmentCarrier["CESKAPOSTAAPI"] = "CESKAPOSTA_API";
    ShipmentCarrier["GORUSH"] = "GORUSH";
    ShipmentCarrier["HOMERUNNER"] = "HOMERUNNER";
    ShipmentCarrier["AMAZONORDER"] = "AMAZON_ORDER";
    ShipmentCarrier["EFWNOWAPI"] = "EFWNOW_API";
    ShipmentCarrier["CBLLOGISTICAAPI"] = "CBL_LOGISTICA_API";
    ShipmentCarrier["NIMBUSPOST"] = "NIMBUSPOST";
    ShipmentCarrier["LOGWINLOGISTICS"] = "LOGWIN_LOGISTICS";
    ShipmentCarrier["NOWLOGAPI"] = "NOWLOG_API";
    ShipmentCarrier["DPDNL"] = "DPD_NL";
    ShipmentCarrier["GODEPENDABLE"] = "GODEPENDABLE";
    ShipmentCarrier["ESDEX"] = "ESDEX";
    ShipmentCarrier["LOGISYSTEMSSFTP"] = "LOGISYSTEMS_SFTP";
    ShipmentCarrier["EXPEDITORS"] = "EXPEDITORS";
    ShipmentCarrier["SNTGLOBALAPI"] = "SNTGLOBAL_API";
    ShipmentCarrier["SHIPX"] = "SHIPX";
    ShipmentCarrier["QINTLAPI"] = "QINTL_API";
    ShipmentCarrier["PACKS"] = "PACKS";
    ShipmentCarrier["POSTNLINTERNATIONAL"] = "POSTNL_INTERNATIONAL";
    ShipmentCarrier["AMAZONEMAILPUSH"] = "AMAZON_EMAIL_PUSH";
    ShipmentCarrier["DHLAPI"] = "DHL_API";
    ShipmentCarrier["SPX"] = "SPX";
    ShipmentCarrier["AXLEHIRE"] = "AXLEHIRE";
    ShipmentCarrier["ICSCOURIER"] = "ICSCOURIER";
    ShipmentCarrier["DIALOGOLOGISTICA"] = "DIALOGO_LOGISTICA";
    ShipmentCarrier["SHUNBANGEXPRESS"] = "SHUNBANG_EXPRESS";
    ShipmentCarrier["TCSAPI"] = "TCS_API";
    ShipmentCarrier["SFEXPRESSCN"] = "SF_EXPRESS_CN";
    ShipmentCarrier["PACKETA"] = "PACKETA";
    ShipmentCarrier["SICTELIWAY"] = "SIC_TELIWAY";
    ShipmentCarrier["MONDIALRELAYFR"] = "MONDIALRELAY_FR";
    ShipmentCarrier["INTIMEFTP"] = "INTIME_FTP";
    ShipmentCarrier["JDEXPRESS"] = "JD_EXPRESS";
    ShipmentCarrier["FASTBOX"] = "FASTBOX";
    ShipmentCarrier["PATHEON"] = "PATHEON";
    ShipmentCarrier["INDIAPOST"] = "INDIA_POST";
    ShipmentCarrier["TIPSAREF"] = "TIPSA_REF";
    ShipmentCarrier["ECOFREIGHT"] = "ECOFREIGHT";
    ShipmentCarrier["VOX"] = "VOX";
    ShipmentCarrier["DIRECTFREIGHTAUREF"] = "DIRECTFREIGHT_AU_REF";
    ShipmentCarrier["BESTTRANSPORTSFTP"] = "BESTTRANSPORT_SFTP";
    ShipmentCarrier["AUSTRALIAPOSTAPI"] = "AUSTRALIA_POST_API";
    ShipmentCarrier["FRAGILEPAKSFTP"] = "FRAGILEPAK_SFTP";
    ShipmentCarrier["FLIPXP"] = "FLIPXP";
    ShipmentCarrier["VALUEWEBHOOK"] = "VALUE_WEBHOOK";
    ShipmentCarrier["DAESHIN"] = "DAESHIN";
    ShipmentCarrier["SHERPA"] = "SHERPA";
    ShipmentCarrier["MWDAPI"] = "MWD_API";
    ShipmentCarrier["SMARTKARGO"] = "SMARTKARGO";
    ShipmentCarrier["DNJEXPRESS"] = "DNJ_EXPRESS";
    ShipmentCarrier["GOPEOPLE"] = "GOPEOPLE";
    ShipmentCarrier["MYSENDLEAPI"] = "MYSENDLE_API";
    ShipmentCarrier["ARAMEXAPI"] = "ARAMEX_API";
    ShipmentCarrier["PIDGE"] = "PIDGE";
    ShipmentCarrier["THAIPARCELS"] = "THAIPARCELS";
    ShipmentCarrier["PANTHERREFERENCEAPI"] = "PANTHER_REFERENCE_API";
    ShipmentCarrier["POSTAPLUS1"] = "POSTAPLUS";
    ShipmentCarrier["BUFFALO"] = "BUFFALO";
    ShipmentCarrier["UENVIOS"] = "U_ENVIOS";
    ShipmentCarrier["ELITECO"] = "ELITE_CO";
    ShipmentCarrier["BARQEXP"] = "BARQEXP";
    ShipmentCarrier["ROCHEINTERNALSFTP"] = "ROCHE_INTERNAL_SFTP";
    ShipmentCarrier["DBSCHENKERICELAND"] = "DBSCHENKER_ICELAND";
    ShipmentCarrier["TNTFRREFERENCE"] = "TNT_FR_REFERENCE";
    ShipmentCarrier["NEWGISTICSAPI"] = "NEWGISTICSAPI";
    ShipmentCarrier["GLOVO"] = "GLOVO";
    ShipmentCarrier["GWLOGISAPI"] = "GWLOGIS_API";
    ShipmentCarrier["SPREETAILAPI"] = "SPREETAIL_API";
    ShipmentCarrier["MOOVA"] = "MOOVA";
    ShipmentCarrier["PLYCONGROUP"] = "PLYCONGROUP";
    ShipmentCarrier["USPSWEBHOOK"] = "USPS_WEBHOOK";
    ShipmentCarrier["REIMAGINEDELIVERY"] = "REIMAGINEDELIVERY";
    ShipmentCarrier["EDFFTP"] = "EDF_FTP";
    ShipmentCarrier["DAO365"] = "DAO365";
    ShipmentCarrier["BIOCAIRFTP"] = "BIOCAIR_FTP";
    ShipmentCarrier["RANSAWEBHOOK"] = "RANSA_WEBHOOK";
    ShipmentCarrier["SHIPXPRES"] = "SHIPXPRES";
    ShipmentCarrier["COURANTPLUSAPI"] = "COURANT_PLUS_API";
    ShipmentCarrier["SHIPA"] = "SHIPA";
    ShipmentCarrier["HOMELOGISTICS"] = "HOMELOGISTICS";
    ShipmentCarrier["DX"] = "DX";
    ShipmentCarrier["POSTEITALIANEPACCOCELERE"] = "POSTE_ITALIANE_PACCOCELERE";
    ShipmentCarrier["TOLLWEBHOOK"] = "TOLL_WEBHOOK";
    ShipmentCarrier["LCTBRAPI"] = "LCTBR_API";
    ShipmentCarrier["DXFREIGHT"] = "DX_FREIGHT";
    ShipmentCarrier["DHLSFTP"] = "DHL_SFTP";
    ShipmentCarrier["SHIPROCKET"] = "SHIPROCKET";
    ShipmentCarrier["UBERWEBHOOK"] = "UBER_WEBHOOK";
    ShipmentCarrier["STATOVERNIGHT"] = "STATOVERNIGHT";
    ShipmentCarrier["BURD"] = "BURD";
    ShipmentCarrier["FASTSHIP"] = "FASTSHIP";
    ShipmentCarrier["IBVENTUREWEBHOOK"] = "IBVENTURE_WEBHOOK";
    ShipmentCarrier["GATIKWEAPI"] = "GATI_KWE_API";
    ShipmentCarrier["CRYOPDPFTP"] = "CRYOPDP_FTP";
    ShipmentCarrier["HUBBED"] = "HUBBED";
    ShipmentCarrier["TIPSAAPI"] = "TIPSA_API";
    ShipmentCarrier["ARASKARGO"] = "ARASKARGO";
    ShipmentCarrier["THIJSNL"] = "THIJS_NL";
    ShipmentCarrier["ATSHEALTHCAREREFERENCE"] = "ATSHEALTHCARE_REFERENCE";
    ShipmentCarrier["Enum99MINUTOS"] = "99MINUTOS";
    ShipmentCarrier["HELLENICPOST"] = "HELLENIC_POST";
    ShipmentCarrier["HSMGLOBAL"] = "HSM_GLOBAL";
    ShipmentCarrier["MNX"] = "MNX";
    ShipmentCarrier["NMTRANSFER"] = "NMTRANSFER";
    ShipmentCarrier["LOGYSTO"] = "LOGYSTO";
    ShipmentCarrier["INDIAPOSTINT"] = "INDIA_POST_INT";
    ShipmentCarrier["AMAZONFBASWISHIPIN"] = "AMAZON_FBA_SWISHIP_IN";
    ShipmentCarrier["SRTTRANSPORT"] = "SRT_TRANSPORT";
    ShipmentCarrier["BOMI"] = "BOMI";
    ShipmentCarrier["DELIVERRSFTP"] = "DELIVERR_SFTP";
    ShipmentCarrier["HSDEXPRESS"] = "HSDEXPRESS";
    ShipmentCarrier["SIMPLETIREWEBHOOK"] = "SIMPLETIRE_WEBHOOK";
    ShipmentCarrier["HUNTEREXPRESSSFTP"] = "HUNTER_EXPRESS_SFTP";
    ShipmentCarrier["UPSAPI"] = "UPS_API";
    ShipmentCarrier["WOOYOUNGLOGISTICSSFTP"] = "WOOYOUNG_LOGISTICS_SFTP";
    ShipmentCarrier["PHSEAPI"] = "PHSE_API";
    ShipmentCarrier["WISHEMAILPUSH"] = "WISH_EMAIL_PUSH";
    ShipmentCarrier["NORTHLINE"] = "NORTHLINE";
    ShipmentCarrier["MEDAFRICA"] = "MEDAFRICA";
    ShipmentCarrier["DPDATSFTP"] = "DPD_AT_SFTP";
    ShipmentCarrier["ANTERAJA"] = "ANTERAJA";
    ShipmentCarrier["DHLGLOBALFORWARDINGAPI"] = "DHL_GLOBAL_FORWARDING_API";
    ShipmentCarrier["LBCEXPRESSAPI"] = "LBCEXPRESS_API";
    ShipmentCarrier["SIMSGLOBAL"] = "SIMSGLOBAL";
    ShipmentCarrier["CDLDELIVERS"] = "CDLDELIVERS";
    ShipmentCarrier["TYP"] = "TYP";
    ShipmentCarrier["TESTINGCOURIERWEBHOOK"] = "TESTING_COURIER_WEBHOOK";
    ShipmentCarrier["PANDAGOAPI"] = "PANDAGO_API";
    ShipmentCarrier["ROYALMAILFTP"] = "ROYAL_MAIL_FTP";
    ShipmentCarrier["THUNDEREXPRESS"] = "THUNDEREXPRESS";
    ShipmentCarrier["SECRETLABWEBHOOK"] = "SECRETLAB_WEBHOOK";
    ShipmentCarrier["SETEL"] = "SETEL";
    ShipmentCarrier["JDWORLDWIDE"] = "JD_WORLDWIDE";
    ShipmentCarrier["DPDRUAPI"] = "DPD_RU_API";
    ShipmentCarrier["ARGENTSWEBHOOK"] = "ARGENTS_WEBHOOK";
    ShipmentCarrier["POSTONE"] = "POSTONE";
    ShipmentCarrier["TUSKLOGISTICS"] = "TUSKLOGISTICS";
    ShipmentCarrier["RHENUSUKAPI"] = "RHENUS_UK_API";
    ShipmentCarrier["TAQBINSGAPI"] = "TAQBIN_SG_API";
    ShipmentCarrier["INNTRALOGSFTP"] = "INNTRALOG_SFTP";
    ShipmentCarrier["DAYROSS"] = "DAYROSS";
    ShipmentCarrier["CORREOSEXPRESSAPI"] = "CORREOSEXPRESS_API";
    ShipmentCarrier["INTERNATIONALSEURAPI"] = "INTERNATIONAL_SEUR_API";
    ShipmentCarrier["YODELAPI"] = "YODEL_API";
    ShipmentCarrier["HEROEXPRESS"] = "HEROEXPRESS";
    ShipmentCarrier["DHLSUPPLYCHAININ"] = "DHL_SUPPLYCHAIN_IN";
    ShipmentCarrier["URGENTCARGUS"] = "URGENT_CARGUS";
    ShipmentCarrier["FRONTDOORCORP"] = "FRONTDOORCORP";
    ShipmentCarrier["JTEXPRESSPH"] = "JTEXPRESS_PH";
    ShipmentCarrier["PARCELSTARSWEBHOOK"] = "PARCELSTARS_WEBHOOK";
    ShipmentCarrier["DPDSKSFTP"] = "DPD_SK_SFTP";
    ShipmentCarrier["MOVIANTO"] = "MOVIANTO";
    ShipmentCarrier["OZEPARTSSHIPPING"] = "OZEPARTS_SHIPPING";
    ShipmentCarrier["KARGOMKOLAY"] = "KARGOMKOLAY";
    ShipmentCarrier["TRUNKRS"] = "TRUNKRS";
    ShipmentCarrier["OMNIRPSWEBHOOK"] = "OMNIRPS_WEBHOOK";
    ShipmentCarrier["CHILEXPRESS"] = "CHILEXPRESS";
    ShipmentCarrier["TESTINGCOURIER"] = "TESTING_COURIER";
    ShipmentCarrier["JNEAPI"] = "JNE_API";
    ShipmentCarrier["BJSHOMEDELIVERYFTP"] = "BJSHOMEDELIVERY_FTP";
    ShipmentCarrier["DEXPRESSWEBHOOK"] = "DEXPRESS_WEBHOOK";
    ShipmentCarrier["USPSAPI"] = "USPS_API";
    ShipmentCarrier["TRANSVIRTUAL"] = "TRANSVIRTUAL";
    ShipmentCarrier["SOLISTICAAPI"] = "SOLISTICA_API";
    ShipmentCarrier["CHIENVENTUREWEBHOOK"] = "CHIENVENTURE_WEBHOOK";
    ShipmentCarrier["DPDUKSFTP"] = "DPD_UK_SFTP";
    ShipmentCarrier["INPOSTUK"] = "INPOST_UK";
    ShipmentCarrier["JAVIT"] = "JAVIT";
    ShipmentCarrier["ZTODOMESTIC"] = "ZTO_DOMESTIC";
    ShipmentCarrier["DHLGTAPI"] = "DHL_GT_API";
    ShipmentCarrier["CEVATRACKING"] = "CEVA_TRACKING";
    ShipmentCarrier["KOMONEXPRESS"] = "KOMON_EXPRESS";
    ShipmentCarrier["EASTWESTCOURIERFTP"] = "EASTWESTCOURIER_FTP";
    ShipmentCarrier["DANNIAO"] = "DANNIAO";
    ShipmentCarrier["SPECTRAN"] = "SPECTRAN";
    ShipmentCarrier["DELIVERIT"] = "DELIVER_IT";
    ShipmentCarrier["RELAISCOLIS"] = "RELAISCOLIS";
    ShipmentCarrier["GLSSPAINAPI"] = "GLS_SPAIN_API";
    ShipmentCarrier["POSTPLUS"] = "POSTPLUS";
    ShipmentCarrier["AIRTERRA"] = "AIRTERRA";
    ShipmentCarrier["GIOECOURIERAPI"] = "GIO_ECOURIER_API";
    ShipmentCarrier["DPDCHSFTP"] = "DPD_CH_SFTP";
    ShipmentCarrier["FEDEXAPI"] = "FEDEX_API";
    ShipmentCarrier["INTERSMARTTRANS"] = "INTERSMARTTRANS";
    ShipmentCarrier["HERMESUKSFTP"] = "HERMES_UK_SFTP";
    ShipmentCarrier["EXELOTFTP"] = "EXELOT_FTP";
    ShipmentCarrier["DHLPAAPI"] = "DHL_PA_API";
    ShipmentCarrier["VIRTRANSPORTSFTP"] = "VIRTRANSPORT_SFTP";
    ShipmentCarrier["WORLDNET"] = "WORLDNET";
    ShipmentCarrier["INSTABOXWEBHOOK"] = "INSTABOX_WEBHOOK";
    ShipmentCarrier["KNG"] = "KNG";
    ShipmentCarrier["FLASHEXPRESSWEBHOOK"] = "FLASHEXPRESS_WEBHOOK";
    ShipmentCarrier["MAGYARPOSTAAPI"] = "MAGYAR_POSTA_API";
    ShipmentCarrier["WESHIPAPI"] = "WESHIP_API";
    ShipmentCarrier["OHIWEBHOOK"] = "OHI_WEBHOOK";
    ShipmentCarrier["MUDITA"] = "MUDITA";
    ShipmentCarrier["BLUEDARTAPI"] = "BLUEDART_API";
    ShipmentCarrier["TCATAPI"] = "T_CAT_API";
    ShipmentCarrier["ADS"] = "ADS";
    ShipmentCarrier["HERMESIT"] = "HERMES_IT";
    ShipmentCarrier["FITZMARKAPI"] = "FITZMARK_API";
    ShipmentCarrier["POSTIAPI"] = "POSTI_API";
    ShipmentCarrier["SMSAEXPRESSWEBHOOK"] = "SMSA_EXPRESS_WEBHOOK";
    ShipmentCarrier["TAMERGROUPWEBHOOK"] = "TAMERGROUP_WEBHOOK";
    ShipmentCarrier["LIVRAPIDE"] = "LIVRAPIDE";
    ShipmentCarrier["NIPPONEXPRESS"] = "NIPPON_EXPRESS";
    ShipmentCarrier["BETTERTRUCKS"] = "BETTERTRUCKS";
    ShipmentCarrier["FAN"] = "FAN";
    ShipmentCarrier["PBUSPSFLATSFTP"] = "PB_USPSFLATS_FTP";
    ShipmentCarrier["PARCELRIGHT"] = "PARCELRIGHT";
    ShipmentCarrier["ITHINKLOGISTICS"] = "ITHINKLOGISTICS";
    ShipmentCarrier["KERRYEXPRESSTHWEBHOOK"] = "KERRY_EXPRESS_TH_WEBHOOK";
    ShipmentCarrier["ECOUTIER"] = "ECOUTIER";
    ShipmentCarrier["SHOWL"] = "SHOWL";
    ShipmentCarrier["BRTITAPI"] = "BRT_IT_API";
    ShipmentCarrier["RIXONHKAPI"] = "RIXONHK_API";
    ShipmentCarrier["DBSCHENKERAPI"] = "DBSCHENKER_API";
    ShipmentCarrier["ILYANGLOGIS"] = "ILYANGLOGIS";
    ShipmentCarrier["MAILBOXETC"] = "MAIL_BOX_ETC";
    ShipmentCarrier["WESHIP"] = "WESHIP";
    ShipmentCarrier["DHLGLOBALMAILAPI"] = "DHL_GLOBAL_MAIL_API";
    ShipmentCarrier["ACTIVOS24API"] = "ACTIVOS24_API";
    ShipmentCarrier["ATSHEALTHCARE"] = "ATSHEALTHCARE";
    ShipmentCarrier["LUWJISTIK"] = "LUWJISTIK";
    ShipmentCarrier["GWWORLD"] = "GW_WORLD";
    ShipmentCarrier["FAIRSENDENAPI"] = "FAIRSENDEN_API";
    ShipmentCarrier["SERVIPWEBHOOK"] = "SERVIP_WEBHOOK";
    ShipmentCarrier["SWISHIP"] = "SWISHIP";
    ShipmentCarrier["TANET"] = "TANET";
    ShipmentCarrier["HOTSINCARGO"] = "HOTSIN_CARGO";
    ShipmentCarrier["DIREX"] = "DIREX";
    ShipmentCarrier["HUANTONG"] = "HUANTONG";
    ShipmentCarrier["IMILEAPI"] = "IMILE_API";
    ShipmentCarrier["BDMNET"] = "BDMNET";
    ShipmentCarrier["AUEXPRESS"] = "AUEXPRESS";
    ShipmentCarrier["NYTLOGISTICS"] = "NYTLOGISTICS";
    ShipmentCarrier["DSVREFERENCE"] = "DSV_REFERENCE";
    ShipmentCarrier["NOVOFARMAWEBHOOK"] = "NOVOFARMA_WEBHOOK";
    ShipmentCarrier["AITWORLDWIDESFTP"] = "AITWORLDWIDE_SFTP";
    ShipmentCarrier["SHOPOLIVE"] = "SHOPOLIVE";
    ShipmentCarrier["FNFZA"] = "FNF_ZA";
    ShipmentCarrier["DHLECOMMERCEGC"] = "DHL_ECOMMERCE_GC";
    ShipmentCarrier["FETCHR"] = "FETCHR";
    ShipmentCarrier["STARLINKSAPI"] = "STARLINKS_API";
    ShipmentCarrier["YYEXPRESS"] = "YYEXPRESS";
    ShipmentCarrier["SERVIENTREGA"] = "SERVIENTREGA";
    ShipmentCarrier["HANJIN"] = "HANJIN";
    ShipmentCarrier["SPANISHSEURFTP"] = "SPANISH_SEUR_FTP";
    ShipmentCarrier["DXB2BCONNUM"] = "DX_B2B_CONNUM";
    ShipmentCarrier["HELTHJEMAPI"] = "HELTHJEM_API";
    ShipmentCarrier["INEXPOST"] = "INEXPOST";
    ShipmentCarrier["A2BBA"] = "A2B_BA";
    ShipmentCarrier["RHENUSGROUP"] = "RHENUS_GROUP";
    ShipmentCarrier["SBERLOGISTICSRU"] = "SBERLOGISTICS_RU";
    ShipmentCarrier["MALCAAMIT"] = "MALCA_AMIT";
    ShipmentCarrier["PPL"] = "PPL";
    ShipmentCarrier["OSMWORLDWIDESFTP"] = "OSM_WORLDWIDE_SFTP";
    ShipmentCarrier["ACILOGISTIX"] = "ACILOGISTIX";
    ShipmentCarrier["OPTIMACOURIER"] = "OPTIMACOURIER";
    ShipmentCarrier["NOVAPOSHTAAPI"] = "NOVA_POSHTA_API";
    ShipmentCarrier["LOGGI"] = "LOGGI";
    ShipmentCarrier["YIFAN"] = "YIFAN";
    ShipmentCarrier["MYDYNALOGIC"] = "MYDYNALOGIC";
    ShipmentCarrier["MORNINGLOBAL"] = "MORNINGLOBAL";
    ShipmentCarrier["CONCISEAPI"] = "CONCISE_API";
    ShipmentCarrier["FXTRAN"] = "FXTRAN";
    ShipmentCarrier["DELIVERYOURPARCELZA"] = "DELIVERYOURPARCEL_ZA";
    ShipmentCarrier["UPARCEL"] = "UPARCEL";
    ShipmentCarrier["MOBIBR"] = "MOBI_BR";
    ShipmentCarrier["LOGINEXTWEBHOOK"] = "LOGINEXT_WEBHOOK";
    ShipmentCarrier["EMS"] = "EMS";
    ShipmentCarrier["SPEEDY"] = "SPEEDY";
})(ShipmentCarrier || (ShipmentCarrier = {}));
/**
 * Schema for ShipmentCarrier
 */
export const shipmentCarrierSchema = stringEnum(ShipmentCarrier, true);
