// @paypal/paypal-server-sdk/dist/cjs/models/aVSCode.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.aVSCodeSchema = exports.AVSCode = void 0;
var schema_1 = require("../schema");
/**
 * Enum for AVSCode
 */
var AVSCode;
(function (AVSCode) {
    AVSCode["A"] = "A";
    AVSCode["B"] = "B";
    AVSCode["C"] = "C";
    AVSCode["D"] = "D";
    AVSCode["E"] = "E";
    AVSCode["F"] = "F";
    AVSCode["G"] = "G";
    AVSCode["I"] = "I";
    AVSCode["M"] = "M";
    AVSCode["N"] = "N";
    AVSCode["P"] = "P";
    AVSCode["R"] = "R";
    AVSCode["S"] = "S";
    AVSCode["U"] = "U";
    AVSCode["W"] = "W";
    AVSCode["X"] = "X";
    AVSCode["Y"] = "Y";
    AVSCode["Z"] = "Z";
    AVSCode["Null"] = "Null";
    AVSCode["Enum0"] = "0";
    AVSCode["Enum1"] = "1";
    AVSCode["Enum2"] = "2";
    AVSCode["Enum3"] = "3";
    AVSCode["Enum4"] = "4";
})(AVSCode = exports.AVSCode || (exports.AVSCode = {}));
/**
 * Schema for AVSCode
 */
exports.aVSCodeSchema = (0, schema_1.stringEnum)(AVSCode, true);
