// @paypal/paypal-server-sdk/dist/cjs/models/bLIKPaymentRequest.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.bLIKPaymentRequestSchema = void 0;
var schema_1 = require("../schema");
var bLIKExperienceContext_1 = require("./bLIKExperienceContext");
var bLIKLevel0PaymentObject_1 = require("./bLIKLevel0PaymentObject");
var bLIKOneClickPaymentRequest_1 = require("./bLIKOneClickPaymentRequest");
exports.bLIKPaymentRequestSchema = (0, schema_1.object)({
    name: ['name', (0, schema_1.string)()],
    countryCode: ['country_code', (0, schema_1.string)()],
    email: ['email', (0, schema_1.optional)((0, schema_1.string)())],
    experienceContext: [
        'experience_context',
        (0, schema_1.optional)((0, schema_1.lazy)(function () { return bLIKExperienceContext_1.bLIKExperienceContextSchema; })),
    ],
    level0: ['level_0', (0, schema_1.optional)((0, schema_1.lazy)(function () { return bLIKLevel0PaymentObject_1.bLIKLevel0PaymentObjectSchema; }))],
    oneClick: [
        'one_click',
        (0, schema_1.optional)((0, schema_1.lazy)(function () { return bLIKOneClickPaymentRequest_1.bLIKOneClickPaymentRequestSchema; })),
    ]
});
