// @paypal/paypal-server-sdk/dist/cjs/models/captureIncompleteReason.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.captureIncompleteReasonSchema = exports.CaptureIncompleteReason = void 0;
var schema_1 = require("../schema");
/**
 * Enum for CaptureIncompleteReason
 */
var CaptureIncompleteReason;
(function (CaptureIncompleteReason) {
    CaptureIncompleteReason["BUYERCOMPLAINT"] = "BUYER_COMPLAINT";
    CaptureIncompleteReason["CHARGEBACK"] = "CHARGEBACK";
    CaptureIncompleteReason["ECHECK"] = "ECHECK";
    CaptureIncompleteReason["INTERNATIONALWITHDRAWAL"] = "INTERNATIONAL_WITHDRAWAL";
    CaptureIncompleteReason["OTHER"] = "OTHER";
    CaptureIncompleteReason["PENDINGREVIEW"] = "PENDING_REVIEW";
    CaptureIncompleteReason["RECEIVINGPREFERENCEMANDATESMANUALACTION"] = "RECEIVING_PREFERENCE_MANDATES_MANUAL_ACTION";
    CaptureIncompleteReason["REFUNDED"] = "REFUNDED";
    CaptureIncompleteReason["TRANSACTIONAPPROVEDAWAITINGFUNDING"] = "TRANSACTION_APPROVED_AWAITING_FUNDING";
    CaptureIncompleteReason["UNILATERAL"] = "UNILATERAL";
    CaptureIncompleteReason["VERIFICATIONREQUIRED"] = "VERIFICATION_REQUIRED";
    CaptureIncompleteReason["DECLINEDBYRISKFRAUDFILTERS"] = "DECLINED_BY_RISK_FRAUD_FILTERS";
})(CaptureIncompleteReason = exports.CaptureIncompleteReason || (exports.CaptureIncompleteReason = {}));
/**
 * Schema for CaptureIncompleteReason
 */
exports.captureIncompleteReasonSchema = (0, schema_1.stringEnum)(CaptureIncompleteReason, true);
