// @paypal/paypal-server-sdk/dist/cjs/models/eCIFlag.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.eCIFlagSchema = exports.ECIFlag = void 0;
var schema_1 = require("../schema");
/**
 * Enum for ECIFlag
 */
var ECIFlag;
(function (ECIFlag) {
    ECIFlag["MASTERCARDNON3DSECURETRANSACTION"] = "MASTERCARD_NON_3D_SECURE_TRANSACTION";
    ECIFlag["MASTERCARDATTEMPTEDAUTHENTICATIONTRANSACTION"] = "MASTERCARD_ATTEMPTED_AUTHENTICATION_TRANSACTION";
    ECIFlag["MASTERCARDFULLYAUTHENTICATEDTRANSACTION"] = "MASTERCARD_FULLY_AUTHENTICATED_TRANSACTION";
    ECIFlag["FULLYAUTHENTICATEDTRANSACTION"] = "FULLY_AUTHENTICATED_TRANSACTION";
    ECIFlag["ATTEMPTEDAUTHENTICATIONTRANSACTION"] = "ATTEMPTED_AUTHENTICATION_TRANSACTION";
    ECIFlag["NON3DSECURETRANSACTION"] = "NON_3D_SECURE_TRANSACTION";
})(ECIFlag = exports.ECIFlag || (exports.ECIFlag = {}));
/**
 * Schema for ECIFlag
 */
exports.eCIFlagSchema = (0, schema_1.stringEnum)(ECIFlag, true);
