// @paypal/paypal-server-sdk/dist/cjs/models/paymentSource.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.paymentSourceSchema = void 0;
var schema_1 = require("../schema");
var applePayRequest_1 = require("./applePayRequest");
var bancontactPaymentRequest_1 = require("./bancontactPaymentRequest");
var bLIKPaymentRequest_1 = require("./bLIKPaymentRequest");
var cardRequest_1 = require("./cardRequest");
var ePSPaymentRequest_1 = require("./ePSPaymentRequest");
var giropayPaymentRequest_1 = require("./giropayPaymentRequest");
var googlePayRequest_1 = require("./googlePayRequest");
var iDEALPaymentRequest_1 = require("./iDEALPaymentRequest");
var myBankPaymentRequest_1 = require("./myBankPaymentRequest");
var p24PaymentRequest_1 = require("./p24PaymentRequest");
var payPalWallet_1 = require("./payPalWallet");
var sofortPaymentRequest_1 = require("./sofortPaymentRequest");
var token_1 = require("./token");
var trustlyPaymentRequest_1 = require("./trustlyPaymentRequest");
var venmoWalletRequest_1 = require("./venmoWalletRequest");
exports.paymentSourceSchema = (0, schema_1.object)({
    card: ['card', (0, schema_1.optional)((0, schema_1.lazy)(function () { return cardRequest_1.cardRequestSchema; }))],
    token: ['token', (0, schema_1.optional)((0, schema_1.lazy)(function () { return token_1.tokenSchema; }))],
    paypal: ['paypal', (0, schema_1.optional)((0, schema_1.lazy)(function () { return payPalWallet_1.payPalWalletSchema; }))],
    bancontact: [
        'bancontact',
        (0, schema_1.optional)((0, schema_1.lazy)(function () { return bancontactPaymentRequest_1.bancontactPaymentRequestSchema; })),
    ],
    blik: ['blik', (0, schema_1.optional)((0, schema_1.lazy)(function () { return bLIKPaymentRequest_1.bLIKPaymentRequestSchema; }))],
    eps: ['eps', (0, schema_1.optional)((0, schema_1.lazy)(function () { return ePSPaymentRequest_1.ePSPaymentRequestSchema; }))],
    giropay: ['giropay', (0, schema_1.optional)((0, schema_1.lazy)(function () { return giropayPaymentRequest_1.giropayPaymentRequestSchema; }))],
    ideal: ['ideal', (0, schema_1.optional)((0, schema_1.lazy)(function () { return iDEALPaymentRequest_1.iDEALPaymentRequestSchema; }))],
    mybank: ['mybank', (0, schema_1.optional)((0, schema_1.lazy)(function () { return myBankPaymentRequest_1.myBankPaymentRequestSchema; }))],
    p24: ['p24', (0, schema_1.optional)((0, schema_1.lazy)(function () { return p24PaymentRequest_1.p24PaymentRequestSchema; }))],
    sofort: ['sofort', (0, schema_1.optional)((0, schema_1.lazy)(function () { return sofortPaymentRequest_1.sofortPaymentRequestSchema; }))],
    trustly: ['trustly', (0, schema_1.optional)((0, schema_1.lazy)(function () { return trustlyPaymentRequest_1.trustlyPaymentRequestSchema; }))],
    applePay: ['apple_pay', (0, schema_1.optional)((0, schema_1.lazy)(function () { return applePayRequest_1.applePayRequestSchema; }))],
    googlePay: ['google_pay', (0, schema_1.optional)((0, schema_1.lazy)(function () { return googlePayRequest_1.googlePayRequestSchema; }))],
    venmo: ['venmo', (0, schema_1.optional)((0, schema_1.lazy)(function () { return venmoWalletRequest_1.venmoWalletRequestSchema; }))]
});
