// @paypal/paypal-server-sdk/dist/cjs/models/paymentSourceResponse.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.paymentSourceResponseSchema = void 0;
var schema_1 = require("../schema");
var applePayPaymentObject_1 = require("./applePayPaymentObject");
var bancontactPaymentObject_1 = require("./bancontactPaymentObject");
var bLIKPaymentObject_1 = require("./bLIKPaymentObject");
var cardResponse_1 = require("./cardResponse");
var ePSPaymentObject_1 = require("./ePSPaymentObject");
var giropayPaymentObject_1 = require("./giropayPaymentObject");
var googlePayWalletResponse_1 = require("./googlePayWalletResponse");
var iDEALPaymentObject_1 = require("./iDEALPaymentObject");
var myBankPaymentObject_1 = require("./myBankPaymentObject");
var p24PaymentObject_1 = require("./p24PaymentObject");
var payPalWalletResponse_1 = require("./payPalWalletResponse");
var sofortPaymentObject_1 = require("./sofortPaymentObject");
var trustlyPaymentObject_1 = require("./trustlyPaymentObject");
var venmoWalletResponse_1 = require("./venmoWalletResponse");
exports.paymentSourceResponseSchema = (0, schema_1.object)({
    card: ['card', (0, schema_1.optional)((0, schema_1.lazy)(function () { return cardResponse_1.cardResponseSchema; }))],
    paypal: ['paypal', (0, schema_1.optional)((0, schema_1.lazy)(function () { return payPalWalletResponse_1.payPalWalletResponseSchema; }))],
    bancontact: [
        'bancontact',
        (0, schema_1.optional)((0, schema_1.lazy)(function () { return bancontactPaymentObject_1.bancontactPaymentObjectSchema; })),
    ],
    blik: ['blik', (0, schema_1.optional)((0, schema_1.lazy)(function () { return bLIKPaymentObject_1.bLIKPaymentObjectSchema; }))],
    eps: ['eps', (0, schema_1.optional)((0, schema_1.lazy)(function () { return ePSPaymentObject_1.ePSPaymentObjectSchema; }))],
    giropay: ['giropay', (0, schema_1.optional)((0, schema_1.lazy)(function () { return giropayPaymentObject_1.giropayPaymentObjectSchema; }))],
    ideal: ['ideal', (0, schema_1.optional)((0, schema_1.lazy)(function () { return iDEALPaymentObject_1.iDEALPaymentObjectSchema; }))],
    mybank: ['mybank', (0, schema_1.optional)((0, schema_1.lazy)(function () { return myBankPaymentObject_1.myBankPaymentObjectSchema; }))],
    p24: ['p24', (0, schema_1.optional)((0, schema_1.lazy)(function () { return p24PaymentObject_1.p24PaymentObjectSchema; }))],
    sofort: ['sofort', (0, schema_1.optional)((0, schema_1.lazy)(function () { return sofortPaymentObject_1.sofortPaymentObjectSchema; }))],
    trustly: ['trustly', (0, schema_1.optional)((0, schema_1.lazy)(function () { return trustlyPaymentObject_1.trustlyPaymentObjectSchema; }))],
    applePay: ['apple_pay', (0, schema_1.optional)((0, schema_1.lazy)(function () { return applePayPaymentObject_1.applePayPaymentObjectSchema; }))],
    googlePay: [
        'google_pay',
        (0, schema_1.optional)((0, schema_1.lazy)(function () { return googlePayWalletResponse_1.googlePayWalletResponseSchema; })),
    ],
    venmo: ['venmo', (0, schema_1.optional)((0, schema_1.lazy)(function () { return venmoWalletResponse_1.venmoWalletResponseSchema; }))]
});
