// @paypal/paypal-server-sdk/dist/cjs/models/payPalWalletExperienceContext.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.payPalWalletExperienceContextSchema = void 0;
var schema_1 = require("../schema");
var payeePaymentMethodPreference_1 = require("./payeePaymentMethodPreference");
var payPalExperienceLandingPage_1 = require("./payPalExperienceLandingPage");
var payPalExperienceUserAction_1 = require("./payPalExperienceUserAction");
var shippingPreference_1 = require("./shippingPreference");
exports.payPalWalletExperienceContextSchema = (0, schema_1.object)({
    brandName: ['brand_name', (0, schema_1.optional)((0, schema_1.string)())],
    locale: ['locale', (0, schema_1.optional)((0, schema_1.string)())],
    shippingPreference: [
        'shipping_preference',
        (0, schema_1.optional)(shippingPreference_1.shippingPreferenceSchema),
    ],
    returnUrl: ['return_url', (0, schema_1.optional)((0, schema_1.string)())],
    cancelUrl: ['cancel_url', (0, schema_1.optional)((0, schema_1.string)())],
    landingPage: ['landing_page', (0, schema_1.optional)(payPalExperienceLandingPage_1.payPalExperienceLandingPageSchema)],
    userAction: ['user_action', (0, schema_1.optional)(payPalExperienceUserAction_1.payPalExperienceUserActionSchema)],
    paymentMethodPreference: [
        'payment_method_preference',
        (0, schema_1.optional)(payeePaymentMethodPreference_1.payeePaymentMethodPreferenceSchema),
    ]
});
