// @paypal/paypal-server-sdk/dist/cjs/models/payPalPaymentTokenUsagePattern.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.payPalPaymentTokenUsagePatternSchema = exports.PayPalPaymentTokenUsagePattern = void 0;
var schema_1 = require("../schema");
/**
 * Enum for PayPalPaymentTokenUsagePattern
 */
var PayPalPaymentTokenUsagePattern;
(function (PayPalPaymentTokenUsagePattern) {
    PayPalPaymentTokenUsagePattern["IMMEDIATE"] = "IMMEDIATE";
    PayPalPaymentTokenUsagePattern["DEFERRED"] = "DEFERRED";
    PayPalPaymentTokenUsagePattern["RECURRINGPREPAID"] = "RECURRING_PREPAID";
    PayPalPaymentTokenUsagePattern["RECURRINGPOSTPAID"] = "RECURRING_POSTPAID";
    PayPalPaymentTokenUsagePattern["THRESHOLDPREPAID"] = "THRESHOLD_PREPAID";
    PayPalPaymentTokenUsagePattern["THRESHOLDPOSTPAID"] = "THRESHOLD_POSTPAID";
})(PayPalPaymentTokenUsagePattern = exports.PayPalPaymentTokenUsagePattern || (exports.PayPalPaymentTokenUsagePattern = {}));
/**
 * Schema for PayPalPaymentTokenUsagePattern
 */
exports.payPalPaymentTokenUsagePatternSchema = (0, schema_1.stringEnum)(PayPalPaymentTokenUsagePattern, true);
