// @paypal/paypal-server-sdk/dist/cjs/models/payPalWalletResponse.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.payPalWalletResponseSchema = void 0;
var schema_1 = require("../schema");
var address_1 = require("./address");
var name_1 = require("./name");
var payPalWalletAccountVerificationStatus_1 = require("./payPalWalletAccountVerificationStatus");
var payPalWalletAttributesResponse_1 = require("./payPalWalletAttributesResponse");
var phoneNumber_1 = require("./phoneNumber");
var phoneType_1 = require("./phoneType");
var taxInfo_1 = require("./taxInfo");
exports.payPalWalletResponseSchema = (0, schema_1.object)({
    emailAddress: ['email_address', (0, schema_1.optional)((0, schema_1.string)())],
    accountId: ['account_id', (0, schema_1.optional)((0, schema_1.string)())],
    accountStatus: [
        'account_status',
        (0, schema_1.optional)(payPalWalletAccountVerificationStatus_1.payPalWalletAccountVerificationStatusSchema),
    ],
    name: ['name', (0, schema_1.optional)((0, schema_1.lazy)(function () { return name_1.nameSchema; }))],
    phoneType: ['phone_type', (0, schema_1.optional)(phoneType_1.phoneTypeSchema)],
    phoneNumber: ['phone_number', (0, schema_1.optional)((0, schema_1.lazy)(function () { return phoneNumber_1.phoneNumberSchema; }))],
    birthDate: ['birth_date', (0, schema_1.optional)((0, schema_1.string)())],
    businessName: ['business_name', (0, schema_1.optional)((0, schema_1.string)())],
    taxInfo: ['tax_info', (0, schema_1.optional)((0, schema_1.lazy)(function () { return taxInfo_1.taxInfoSchema; }))],
    address: ['address', (0, schema_1.optional)((0, schema_1.lazy)(function () { return address_1.addressSchema; }))],
    attributes: [
        'attributes',
        (0, schema_1.optional)((0, schema_1.lazy)(function () { return payPalWalletAttributesResponse_1.payPalWalletAttributesResponseSchema; })),
    ]
});
