// @paypal/paypal-server-sdk/dist/cjs/models/payPalWalletVaultInstruction.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.payPalWalletVaultInstructionSchema = void 0;
var schema_1 = require("../schema");
var payPalPaymentTokenCustomerType_1 = require("./payPalPaymentTokenCustomerType");
var payPalPaymentTokenUsagePattern_1 = require("./payPalPaymentTokenUsagePattern");
var payPalPaymentTokenUsageType_1 = require("./payPalPaymentTokenUsageType");
var storeInVaultInstruction_1 = require("./storeInVaultInstruction");
exports.payPalWalletVaultInstructionSchema = (0, schema_1.object)({
    storeInVault: ['store_in_vault', (0, schema_1.optional)(storeInVaultInstruction_1.storeInVaultInstructionSchema)],
    description: ['description', (0, schema_1.optional)((0, schema_1.string)())],
    usagePattern: [
        'usage_pattern',
        (0, schema_1.optional)(payPalPaymentTokenUsagePattern_1.payPalPaymentTokenUsagePatternSchema),
    ],
    usageType: ['usage_type', payPalPaymentTokenUsageType_1.payPalPaymentTokenUsageTypeSchema],
    customerType: [
        'customer_type',
        (0, schema_1.optional)(payPalPaymentTokenCustomerType_1.payPalPaymentTokenCustomerTypeSchema),
    ],
    permitMultiplePaymentTokens: [
        'permit_multiple_payment_tokens',
        (0, schema_1.optional)((0, schema_1.boolean)()),
    ]
});
