// @paypal/paypal-server-sdk/dist/cjs/models/processorResponseCode.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.processorResponseCodeSchema = exports.ProcessorResponseCode = void 0;
var schema_1 = require("../schema");
/**
 * Enum for ProcessorResponseCode
 */
var ProcessorResponseCode;
(function (ProcessorResponseCode) {
    ProcessorResponseCode["Enum0000"] = "0000";
    ProcessorResponseCode["Enum00N7"] = "00N7";
    ProcessorResponseCode["Enum0100"] = "0100";
    ProcessorResponseCode["Enum0390"] = "0390";
    ProcessorResponseCode["Enum0500"] = "0500";
    ProcessorResponseCode["Enum0580"] = "0580";
    ProcessorResponseCode["Enum0800"] = "0800";
    ProcessorResponseCode["Enum0880"] = "0880";
    ProcessorResponseCode["Enum0890"] = "0890";
    ProcessorResponseCode["Enum0960"] = "0960";
    ProcessorResponseCode["Enum0R00"] = "0R00";
    ProcessorResponseCode["Enum1000"] = "1000";
    ProcessorResponseCode["Enum10BR"] = "10BR";
    ProcessorResponseCode["Enum1300"] = "1300";
    ProcessorResponseCode["Enum1310"] = "1310";
    ProcessorResponseCode["Enum1312"] = "1312";
    ProcessorResponseCode["Enum1317"] = "1317";
    ProcessorResponseCode["Enum1320"] = "1320";
    ProcessorResponseCode["Enum1330"] = "1330";
    ProcessorResponseCode["Enum1335"] = "1335";
    ProcessorResponseCode["Enum1340"] = "1340";
    ProcessorResponseCode["Enum1350"] = "1350";
    ProcessorResponseCode["Enum1352"] = "1352";
    ProcessorResponseCode["Enum1360"] = "1360";
    ProcessorResponseCode["Enum1370"] = "1370";
    ProcessorResponseCode["Enum1380"] = "1380";
    ProcessorResponseCode["Enum1382"] = "1382";
    ProcessorResponseCode["Enum1384"] = "1384";
    ProcessorResponseCode["Enum1390"] = "1390";
    ProcessorResponseCode["Enum1393"] = "1393";
    ProcessorResponseCode["Enum5100"] = "5100";
    ProcessorResponseCode["Enum5110"] = "5110";
    ProcessorResponseCode["Enum5120"] = "5120";
    ProcessorResponseCode["Enum5130"] = "5130";
    ProcessorResponseCode["Enum5135"] = "5135";
    ProcessorResponseCode["Enum5140"] = "5140";
    ProcessorResponseCode["Enum5150"] = "5150";
    ProcessorResponseCode["Enum5160"] = "5160";
    ProcessorResponseCode["Enum5170"] = "5170";
    ProcessorResponseCode["Enum5180"] = "5180";
    ProcessorResponseCode["Enum5190"] = "5190";
    ProcessorResponseCode["Enum5200"] = "5200";
    ProcessorResponseCode["Enum5210"] = "5210";
    ProcessorResponseCode["Enum5400"] = "5400";
    ProcessorResponseCode["Enum5500"] = "5500";
    ProcessorResponseCode["Enum5650"] = "5650";
    ProcessorResponseCode["Enum5700"] = "5700";
    ProcessorResponseCode["Enum5710"] = "5710";
    ProcessorResponseCode["Enum5800"] = "5800";
    ProcessorResponseCode["Enum5900"] = "5900";
    ProcessorResponseCode["Enum5910"] = "5910";
    ProcessorResponseCode["Enum5920"] = "5920";
    ProcessorResponseCode["Enum5930"] = "5930";
    ProcessorResponseCode["Enum5950"] = "5950";
    ProcessorResponseCode["Enum6300"] = "6300";
    ProcessorResponseCode["Enum7600"] = "7600";
    ProcessorResponseCode["Enum7700"] = "7700";
    ProcessorResponseCode["Enum7710"] = "7710";
    ProcessorResponseCode["Enum7800"] = "7800";
    ProcessorResponseCode["Enum7900"] = "7900";
    ProcessorResponseCode["Enum8000"] = "8000";
    ProcessorResponseCode["Enum8010"] = "8010";
    ProcessorResponseCode["Enum8020"] = "8020";
    ProcessorResponseCode["Enum8030"] = "8030";
    ProcessorResponseCode["Enum8100"] = "8100";
    ProcessorResponseCode["Enum8110"] = "8110";
    ProcessorResponseCode["Enum8220"] = "8220";
    ProcessorResponseCode["Enum9100"] = "9100";
    ProcessorResponseCode["Enum9500"] = "9500";
    ProcessorResponseCode["Enum9510"] = "9510";
    ProcessorResponseCode["Enum9520"] = "9520";
    ProcessorResponseCode["Enum9530"] = "9530";
    ProcessorResponseCode["Enum9540"] = "9540";
    ProcessorResponseCode["Enum9600"] = "9600";
    ProcessorResponseCode["PCNR"] = "PCNR";
    ProcessorResponseCode["PCVV"] = "PCVV";
    ProcessorResponseCode["PP06"] = "PP06";
    ProcessorResponseCode["PPRN"] = "PPRN";
    ProcessorResponseCode["PPAD"] = "PPAD";
    ProcessorResponseCode["PPAB"] = "PPAB";
    ProcessorResponseCode["PPAE"] = "PPAE";
    ProcessorResponseCode["PPAG"] = "PPAG";
    ProcessorResponseCode["PPAI"] = "PPAI";
    ProcessorResponseCode["PPAR"] = "PPAR";
    ProcessorResponseCode["PPAU"] = "PPAU";
    ProcessorResponseCode["PPAV"] = "PPAV";
    ProcessorResponseCode["PPAX"] = "PPAX";
    ProcessorResponseCode["PPBG"] = "PPBG";
    ProcessorResponseCode["PPC2"] = "PPC2";
    ProcessorResponseCode["PPCE"] = "PPCE";
    ProcessorResponseCode["PPCO"] = "PPCO";
    ProcessorResponseCode["PPCR"] = "PPCR";
    ProcessorResponseCode["PPCT"] = "PPCT";
    ProcessorResponseCode["PPCU"] = "PPCU";
    ProcessorResponseCode["PPD3"] = "PPD3";
    ProcessorResponseCode["PPDC"] = "PPDC";
    ProcessorResponseCode["PPDI"] = "PPDI";
    ProcessorResponseCode["PPDV"] = "PPDV";
    ProcessorResponseCode["PPDT"] = "PPDT";
    ProcessorResponseCode["PPEF"] = "PPEF";
    ProcessorResponseCode["PPEL"] = "PPEL";
    ProcessorResponseCode["PPER"] = "PPER";
    ProcessorResponseCode["PPEX"] = "PPEX";
    ProcessorResponseCode["PPFE"] = "PPFE";
    ProcessorResponseCode["PPFI"] = "PPFI";
    ProcessorResponseCode["PPFR"] = "PPFR";
    ProcessorResponseCode["PPFV"] = "PPFV";
    ProcessorResponseCode["PPGR"] = "PPGR";
    ProcessorResponseCode["PPH1"] = "PPH1";
    ProcessorResponseCode["PPIF"] = "PPIF";
    ProcessorResponseCode["PPII"] = "PPII";
    ProcessorResponseCode["PPIM"] = "PPIM";
    ProcessorResponseCode["PPIT"] = "PPIT";
    ProcessorResponseCode["PPLR"] = "PPLR";
    ProcessorResponseCode["PPLS"] = "PPLS";
    ProcessorResponseCode["PPMB"] = "PPMB";
    ProcessorResponseCode["PPMC"] = "PPMC";
    ProcessorResponseCode["PPMD"] = "PPMD";
    ProcessorResponseCode["PPNC"] = "PPNC";
    ProcessorResponseCode["PPNL"] = "PPNL";
    ProcessorResponseCode["PPNM"] = "PPNM";
    ProcessorResponseCode["PPNT"] = "PPNT";
    ProcessorResponseCode["PPPH"] = "PPPH";
    ProcessorResponseCode["PPPI"] = "PPPI";
    ProcessorResponseCode["PPPM"] = "PPPM";
    ProcessorResponseCode["PPQC"] = "PPQC";
    ProcessorResponseCode["PPRE"] = "PPRE";
    ProcessorResponseCode["PPRF"] = "PPRF";
    ProcessorResponseCode["PPRR"] = "PPRR";
    ProcessorResponseCode["PPS0"] = "PPS0";
    ProcessorResponseCode["PPS1"] = "PPS1";
    ProcessorResponseCode["PPS2"] = "PPS2";
    ProcessorResponseCode["PPS3"] = "PPS3";
    ProcessorResponseCode["PPS4"] = "PPS4";
    ProcessorResponseCode["PPS5"] = "PPS5";
    ProcessorResponseCode["PPS6"] = "PPS6";
    ProcessorResponseCode["PPSC"] = "PPSC";
    ProcessorResponseCode["PPSD"] = "PPSD";
    ProcessorResponseCode["PPSE"] = "PPSE";
    ProcessorResponseCode["PPTE"] = "PPTE";
    ProcessorResponseCode["PPTF"] = "PPTF";
    ProcessorResponseCode["PPTI"] = "PPTI";
    ProcessorResponseCode["PPTR"] = "PPTR";
    ProcessorResponseCode["PPTT"] = "PPTT";
    ProcessorResponseCode["PPTV"] = "PPTV";
    ProcessorResponseCode["PPUA"] = "PPUA";
    ProcessorResponseCode["PPUC"] = "PPUC";
    ProcessorResponseCode["PPUE"] = "PPUE";
    ProcessorResponseCode["PPUI"] = "PPUI";
    ProcessorResponseCode["PPUP"] = "PPUP";
    ProcessorResponseCode["PPUR"] = "PPUR";
    ProcessorResponseCode["PPVC"] = "PPVC";
    ProcessorResponseCode["PPVE"] = "PPVE";
    ProcessorResponseCode["PPVT"] = "PPVT";
})(ProcessorResponseCode = exports.ProcessorResponseCode || (exports.ProcessorResponseCode = {}));
/**
 * Schema for ProcessorResponseCode
 */
exports.processorResponseCodeSchema = (0, schema_1.stringEnum)(ProcessorResponseCode, true);
