// @paypal/paypal-server-sdk/dist/cjs/models/venmoPaymentTokenUsagePattern.js
"use strict";
/**
 * Paypal Server SDKLib
 *
 * This file was automatically generated by APIMATIC v3.0 ( https://www.apimatic.io ).
 */
exports.__esModule = true;
exports.venmoPaymentTokenUsagePatternSchema = exports.VenmoPaymentTokenUsagePattern = void 0;
var schema_1 = require("../schema");
/**
 * Enum for VenmoPaymentTokenUsagePattern
 */
var VenmoPaymentTokenUsagePattern;
(function (VenmoPaymentTokenUsagePattern) {
    VenmoPaymentTokenUsagePattern["IMMEDIATE"] = "IMMEDIATE";
    VenmoPaymentTokenUsagePattern["DEFERRED"] = "DEFERRED";
    VenmoPaymentTokenUsagePattern["RECURRINGPREPAID"] = "RECURRING_PREPAID";
    VenmoPaymentTokenUsagePattern["RECURRINGPOSTPAID"] = "RECURRING_POSTPAID";
    VenmoPaymentTokenUsagePattern["THRESHOLDPREPAID"] = "THRESHOLD_PREPAID";
    VenmoPaymentTokenUsagePattern["THRESHOLDPOSTPAID"] = "THRESHOLD_POSTPAID";
})(VenmoPaymentTokenUsagePattern = exports.VenmoPaymentTokenUsagePattern || (exports.VenmoPaymentTokenUsagePattern = {}));
/**
 * Schema for VenmoPaymentTokenUsagePattern
 */
exports.venmoPaymentTokenUsagePatternSchema = (0, schema_1.stringEnum)(VenmoPaymentTokenUsagePattern, true);
